<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <meta http-equiv="Content-Style-Type" content="text/css" />
        <meta name="generator" content="pandoc" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
                        <title>01_title_page</title>
        <style type="text/css">code{white-space: pre;}</style>
                                                    <style>
            body {
                font-family: Georgia;
                max-width: 800px;
                margin: 0 auto;
                line-height: 30px;
                font-size: 18px;
                padding-left: 350px;
                padding-right: 50px;
                color: #111;
            }
            
            h1, h2, h3, h4, h5, h6 {
                font-family: Arial;
            }
            
            h1 {
                padding-top: 200px;
                line-height: 50px;
            }
            h2 {
                padding-top: 30px;
            }
            h3 {
                padding-top: 20px;
            }
            h4 {
                padding-top: 10px;
            }
            p {
                text-align: justify;
            }
            p a {
                word-wrap: break-word;
                white-space: pre;
            }
            code {
                word-wrap: break-word;
            }
            blockquote {
                border-left: 3px solid #eee;
                margin-left: 20px;
                padding-left: 20px;
            }
            
            ::selection {
                background-color: #E4E4E4;
            }
            
            table {
                width: 100%;
            }
            table caption {
                font-weight: bold;
            }
            table tr {
                padding: 0;
                margin: 0;
                background-color: #f0f0f0;
            }
            table tr.even {
                background-color: #fafafa;
            }
            table td {
                margin: 0;
                padding: 3px 5px;
            }
            
            p span.added {
                color: green;
                background-color: #FFF3C5;
            }
            p span.removed {
                color: red;
                background-color: #FFF3C5;
            }
            
            #title-page {
                padding: 80px 0;
            }
            
            #TOC {
                position: fixed;
                left: 0;
                top: 0;
                overflow-y: scroll;
                height: 100%;
                background: #fafafa;
                max-width: 300px;
                font-family: Arial;
                font-size: 15px;
                line-height: 30px;
            }
            ::-webkit-scrollbar {
                width: 8px;
            }
            ::-webkit-scrollbar-track {
                background-color: #ECECEC;
            }
            ::-webkit-scrollbar-thumb {
                background-color: #B0B0B0;
                border-radius: 8px;
            }
            #TOC > ul {
                padding-right: 10px;
            }
            #TOC ul {
                list-style: none;
                padding-left: 20px;
            }
            #TOC ul li a {
                text-decoration: none;
                color: #364149;
                text-overflow: ellipsis;
                display: block;
                white-space: nowrap;
                overflow: hidden;
            }
            #TOC ul li a:hover {
                color: #008cff;
            }
            
            .figure {
                text-align: center;
            }
            .figure p {
                text-align: center;
                font-style: italic;
            }
            .figure img {
                  width: 100%;
            }
            </style>
                <script src="js/jquery.js"></script>
        <script src="js/diff.js"></script>
        <script src="js/main.js"></script>
    </head>
    <body>
                <!--
                -->
        <div id="title-page">
            <h1>This is the title of the thesis</h1>
            <h2>Firstname Surname</h2>
        </div>
                    <div id="TOC">
                <ul>
                <li><a href="#abstract">Abstract</a></li>
                <li><a href="#acknowledgements">Acknowledgements</a></li>
                <li><a href="#list-of-figures">List of figures</a></li>
                <li><a href="#list-of-tables">List of tables</a></li>
                <li><a href="#abbreviations">Abbreviations</a></li>
                <li><a href="#simple-scalable-proteomic-imaging-for-high-dimensional-profiling-of-intact-systems"><span class="toc-section-number">1</span> Simple, Scalable Proteomic Imaging for High-Dimensional Profiling of Intact Systems</a><ul>
                <li><a href="#summary"><span class="toc-section-number">1.1</span> Summary</a></li>
                <li><a href="#introduction"><span class="toc-section-number">1.2</span> Introduction</a></li>
                <li><a href="#results"><span class="toc-section-number">1.3</span> Results</a><ul>
                <li><a href="#synchronizing-dialdehyde-tissue-gel-formationenables-scalable-tissue-preservation"><span class="toc-section-number">1.3.1</span> Synchronizing Dialdehyde-Tissue-Gel FormationEnables Scalable Tissue Preservation</a></li>
                <li><a href="#dialdehyde-tissue-gel-preserves-structural-andmolecular-information-effectively"><span class="toc-section-number">1.3.2</span> Dialdehyde-Tissue-Gel Preserves Structural andMolecular Information Effectively</a></li>
                <li><a href="#switch-and-robust-computational-algorithms-enablehighly-multiplexed-imaging-at-single-cell-resolution"><span class="toc-section-number">1.3.3</span> SWITCH and Robust Computational Algorithms EnableHighly Multiplexed Imaging at Single-Cell Resolution</a></li>
                <li><a href="#switch-enables-simple-rapid-and-scalable-tissue-clearing"><span class="toc-section-number">1.3.4</span> SWITCH Enables Simple, Rapid, and Scalable Tissue-Clearing</a></li>
                <li><a href="#switch-enables-visualization-and-quantitative-analysisof-entire-myelinated-fiber-tracts"><span class="toc-section-number">1.3.5</span> SWITCH Enables Visualization and Quantitative Analysisof Entire Myelinated Fiber Tracts</a></li>
                <li><a href="#switch-enables-scalable-and-uniform-antibody-labeling"><span class="toc-section-number">1.3.6</span> SWITCH Enables Scalable and Uniform Antibody Labeling</a></li>
                </ul></li>
                <li><a href="#discussion"><span class="toc-section-number">1.4</span> Discussion</a></li>
                <li><a href="#experimental-procedures"><span class="toc-section-number">1.5</span> Experimental Procedures</a><ul>
                <li><a href="#mice"><span class="toc-section-number">1.5.1</span> Mice</a></li>
                <li><a href="#switch-mediated-tissue-preservation"><span class="toc-section-number">1.5.2</span> SWITCH-Mediated Tissue Preservation</a></li>
                <li><a href="#passive-clearing-with-thermal-energy"><span class="toc-section-number">1.5.3</span> Passive Clearing with Thermal Energy</a></li>
                <li><a href="#sample-delabeling"><span class="toc-section-number">1.5.4</span> Sample Delabeling</a></li>
                <li><a href="#switch-mediated-fluorescent-labeling"><span class="toc-section-number">1.5.5</span> SWITCH-Mediated Fluorescent Labeling</a></li>
                </ul></li>
                <li><a href="#supplemental-information"><span class="toc-section-number">1.6</span> Supplemental Information</a></li>
                <li><a href="#author-contributions"><span class="toc-section-number">1.7</span> Author Contributions</a></li>
                <li><a href="#acknowledgments"><span class="toc-section-number">1.8</span> Acknowledgments</a></li>
                <li><a href="#references"><span class="toc-section-number">1.9</span> References</a></li>
                </ul></li>
                <li><a href="#ultrafast-immunostaining-of-organ-scale-tissues-for-scalable-proteomic-phenotyping"><span class="toc-section-number">2</span> Ultrafast immunostaining of organ-scale tissues for scalable proteomic phenotyping</a><ul>
                <li><a href="#abstract-1"><span class="toc-section-number">2.1</span> Abstract</a></li>
                <li><a href="#introduction-1"><span class="toc-section-number">2.2</span> Introduction</a></li>
                <li><a href="#results-1"><span class="toc-section-number">2.3</span> Results</a><ul>
                <li><a href="#eflash-mechanism"><span class="toc-section-number">2.3.1</span> eFLASH mechanism</a></li>
                <li><a href="#universal-applicability-of-eflash"><span class="toc-section-number">2.3.2</span> Universal applicability of eFLASH</a></li>
                <li><a href="#a-quantitative-brain-wide-cell-type-mapping-with-eflash"><span class="toc-section-number">2.3.3</span> A quantitative, brain-wide cell type mapping with eFLASH</a></li>
                <li><a href="#brain-wide-comparison-of-genetic-and-protein-based-cell-type-labeling"><span class="toc-section-number">2.3.4</span> Brain-wide comparison of genetic and protein-based cell type labeling</a></li>
                <li><a href="#multidimensional-single-cell-analysis-of-marmoset-visual-cortex"><span class="toc-section-number">2.3.5</span> Multidimensional single-cell analysis of marmoset visual cortex</a></li>
                </ul></li>
                <li><a href="#discussion-1"><span class="toc-section-number">2.4</span> Discussion</a></li>
                <li><a href="#acknowledgements-1"><span class="toc-section-number">2.5</span> Acknowledgements</a></li>
                <li><a href="#author-contributions-1"><span class="toc-section-number">2.6</span> Author Contributions</a></li>
                <li><a href="#competing-interests"><span class="toc-section-number">2.7</span> Competing Interests</a></li>
                <li><a href="#references-1"><span class="toc-section-number">2.8</span> References</a></li>
                <li><a href="#methods"><span class="toc-section-number">2.9</span> Methods</a><ul>
                <li><a href="#mice-1"><span class="toc-section-number">2.9.1</span> Mice</a></li>
                <li><a href="#marmoset"><span class="toc-section-number">2.9.2</span> Marmoset</a></li>
                <li><a href="#organoids"><span class="toc-section-number">2.9.3</span> Organoids</a></li>
                <li><a href="#contextual-fear-conditioning"><span class="toc-section-number">2.9.4</span> Contextual fear conditioning</a></li>
                <li><a href="#sodium-deoxycholate-nadc-concentration-measurement"><span class="toc-section-number">2.9.5</span> Sodium Deoxycholate (NaDC) Concentration Measurement</a></li>
                <li><a href="#shield-processing"><span class="toc-section-number">2.9.6</span> SHIELD Processing</a></li>
                <li><a href="#passive-clearing-delipidation"><span class="toc-section-number">2.9.7</span> Passive Clearing (Delipidation)</a></li>
                <li><a href="#active-clearing-stochastic-electrotransport"><span class="toc-section-number">2.9.8</span> Active Clearing (Stochastic Electrotransport)</a></li>
                <li><a href="#antibody-delabeling"><span class="toc-section-number">2.9.9</span> Antibody Delabeling</a></li>
                <li><a href="#passive-immunohistochemistry"><span class="toc-section-number">2.9.10</span> Passive Immunohistochemistry</a></li>
                <li><a href="#eflash-protocol"><span class="toc-section-number">2.9.11</span> eFLASH Protocol</a></li>
                <li><a href="#dye-conjugation-of-secondary-antibodies"><span class="toc-section-number">2.9.12</span> Dye Conjugation of Secondary Antibodies</a></li>
                <li><a href="#refractive-index-matching"><span class="toc-section-number">2.9.13</span> Refractive Index Matching</a></li>
                <li><a href="#fixation-of-labeled-samples"><span class="toc-section-number">2.9.14</span> Fixation of Labeled Samples</a></li>
                <li><a href="#light-sheet-imaging-and-post-processing"><span class="toc-section-number">2.9.15</span> Light-sheet Imaging and Post-processing</a></li>
                <li><a href="#cell-detection"><span class="toc-section-number">2.9.16</span> Cell Detection</a></li>
                <li><a href="#atlas-alignment"><span class="toc-section-number">2.9.17</span> Atlas Alignment</a></li>
                <li><a href="#brain-region-segmenation"><span class="toc-section-number">2.9.18</span> Brain Region Segmenation</a></li>
                <li><a href="#manual-image-analysis"><span class="toc-section-number">2.9.19</span> Manual Image Analysis</a></li>
                <li><a href="#code-availability"><span class="toc-section-number">2.9.20</span> Code Availability</a></li>
                <li><a href="#data-availability"><span class="toc-section-number">2.9.21</span> Data Availability</a></li>
                </ul></li>
                </ul></li>
                <li><a href="#multiplexed-and-scalable-super-resolution-imaging-of-three-dimensional-protein-localization-in-size-adjustable-tissues"><span class="toc-section-number">3</span> Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues</a><ul>
                <li><a href="#summary-1"><span class="toc-section-number">3.1</span> Summary</a></li>
                <li><a href="#introduction-2"><span class="toc-section-number">3.2</span> Introduction</a></li>
                <li><a href="#results-2"><span class="toc-section-number">3.3</span> Results</a><ul>
                <li><a href="#organ-scale-tissue-expansion-without-loss-of-proteins"><span class="toc-section-number">3.3.1</span> Organ-scale tissue expansion without loss of proteins</a></li>
                <li><a href="#preservation-of-multiscale-architectures"><span class="toc-section-number">3.3.2</span> Preservation of multiscale architectures</a></li>
                <li><a href="#labeling-proteome-with-conventional-antibodies"><span class="toc-section-number">3.3.3</span> Labeling proteome with conventional antibodies</a></li>
                <li><a href="#super-resolution-imaging-of-3d-subcellular-architectures"><span class="toc-section-number">3.3.4</span> Super-resolution imaging of 3D subcellular architectures</a></li>
                <li><a href="#highly-multiplexed-immunolabeling"><span class="toc-section-number">3.3.5</span> Highly multiplexed immunolabeling</a></li>
                <li><a href="#maping-neural-projections"><span class="toc-section-number">3.3.6</span> MAPing neural projections</a></li>
                <li><a href="#immunolabeling-and-imaging-of-mm-thick-tissues"><span class="toc-section-number">3.3.7</span> Immunolabeling and imaging of mm-thick tissues</a></li>
                </ul></li>
                <li><a href="#discussion-2"><span class="toc-section-number">3.4</span> Discussion</a></li>
                <li><a href="#methods-1"><span class="toc-section-number">3.5</span> Methods</a></li>
                <li><a href="#acknowledgements-2"><span class="toc-section-number">3.6</span> Acknowledgements</a></li>
                <li><a href="#author-contributions-2"><span class="toc-section-number">3.7</span> Author Contributions</a></li>
                <li><a href="#competing-financial-interests"><span class="toc-section-number">3.8</span> Competing Financial Interests</a></li>
                <li><a href="#references-2"><span class="toc-section-number">3.9</span> References</a></li>
                <li><a href="#methods-2"><span class="toc-section-number">3.10</span> Methods</a><ul>
                <li><a href="#bsa-hydrogel-denaturation-and-expansion"><span class="toc-section-number">3.10.1</span> BSA hydrogel denaturation and expansion</a></li>
                <li><a href="#general-map-protocol"><span class="toc-section-number">3.10.2</span> General MAP protocol</a></li>
                <li><a href="#expansion-according-to-various-aa-and-sa-concentrations"><span class="toc-section-number">3.10.3</span> Expansion according to various AA and SA concentrations</a></li>
                <li><a href="#shrinkage-and-ri-matching"><span class="toc-section-number">3.10.4</span> Shrinkage and RI matching</a></li>
                <li><a href="#map-processing-of-cerebral-organoid"><span class="toc-section-number">3.10.5</span> MAP processing of cerebral organoid</a></li>
                <li><a href="#cultured-cell-experiment"><span class="toc-section-number">3.10.6</span> Cultured cell experiment</a></li>
                <li><a href="#immunostaining-of-brain-tissue"><span class="toc-section-number">3.10.7</span> Immunostaining of brain tissue</a></li>
                <li><a href="#mounting-and-imaging"><span class="toc-section-number">3.10.8</span> Mounting and imaging</a></li>
                <li><a href="#large-tissue-staining"><span class="toc-section-number">3.10.9</span> Large tissue staining</a></li>
                <li><a href="#cell-distortion-analysis"><span class="toc-section-number">3.10.10</span> Cell distortion analysis</a></li>
                <li><a href="#tissue-distortion-analysis"><span class="toc-section-number">3.10.11</span> Tissue distortion analysis</a></li>
                <li><a href="#neurofilament-tracing"><span class="toc-section-number">3.10.12</span> Neurofilament tracing</a></li>
                <li><a href="#synaptic-and-fiber-intensity-profiles"><span class="toc-section-number">3.10.13</span> Synaptic and fiber intensity profiles</a></li>
                </ul></li>
                </ul></li>
                <li><a href="#research-containing-a-figure"><span class="toc-section-number">4</span> Research containing a figure</a><ul>
                <li><a href="#introduction-3"><span class="toc-section-number">4.1</span> Introduction</a></li>
                <li><a href="#method"><span class="toc-section-number">4.2</span> Method</a><ul>
                <li><a href="#subsection-1"><span class="toc-section-number">4.2.1</span> Subsection 1</a></li>
                <li><a href="#subsection-2"><span class="toc-section-number">4.2.2</span> Subsection 2</a></li>
                </ul></li>
                <li><a href="#results-3"><span class="toc-section-number">4.3</span> Results</a></li>
                <li><a href="#discussion-3"><span class="toc-section-number">4.4</span> Discussion</a></li>
                <li><a href="#conclusion"><span class="toc-section-number">4.5</span> Conclusion</a></li>
                </ul></li>
                <li><a href="#research-containing-a-table"><span class="toc-section-number">5</span> Research containing a table</a><ul>
                <li><a href="#introduction-4"><span class="toc-section-number">5.1</span> Introduction</a></li>
                <li><a href="#method-1"><span class="toc-section-number">5.2</span> Method</a><ul>
                <li><a href="#subsection-1-1"><span class="toc-section-number">5.2.1</span> Subsection 1</a></li>
                <li><a href="#subsection-2-1"><span class="toc-section-number">5.2.2</span> Subsection 2</a></li>
                </ul></li>
                <li><a href="#results-4"><span class="toc-section-number">5.3</span> Results</a></li>
                <li><a href="#discussion-4"><span class="toc-section-number">5.4</span> Discussion</a></li>
                <li><a href="#conclusion-1"><span class="toc-section-number">5.5</span> Conclusion</a></li>
                </ul></li>
                <li><a href="#final-research-study"><span class="toc-section-number">6</span> Final research study</a><ul>
                <li><a href="#introduction-5"><span class="toc-section-number">6.1</span> Introduction</a></li>
                <li><a href="#method-2"><span class="toc-section-number">6.2</span> Method</a><ul>
                <li><a href="#subsection-1-2"><span class="toc-section-number">6.2.1</span> Subsection 1</a></li>
                <li><a href="#subsection-2-2"><span class="toc-section-number">6.2.2</span> Subsection 2</a></li>
                </ul></li>
                <li><a href="#results-5"><span class="toc-section-number">6.3</span> Results</a></li>
                <li><a href="#discussion-5"><span class="toc-section-number">6.4</span> Discussion</a></li>
                <li><a href="#conclusion-2"><span class="toc-section-number">6.5</span> Conclusion</a></li>
                </ul></li>
                <li><a href="#conclusion-3"><span class="toc-section-number">7</span> Conclusion</a><ul>
                <li><a href="#thesis-summary"><span class="toc-section-number">7.1</span> Thesis summary</a></li>
                <li><a href="#future-work"><span class="toc-section-number">7.2</span> Future work</a></li>
                </ul></li>
                <li><a href="#appendix-1-some-extra-stuff">Appendix 1: Some extra stuff</a></li>
                <li><a href="#appendix-2-some-more-extra-stuff">Appendix 2: Some more extra stuff</a></li>
                <li><a href="#references-3">References</a></li>
                </ul>
            </div>
                                
<!-- This page is for an official declaration. -->

<p>   </p>
<h1 id="abstract" class="unnumbered">Abstract</h1>
<p>The brain is the most complex human organ, containing components from the nanometer scale to the centimeter scale, such as synapses, neurons, and brain regions. However, many experimental techniques in imaging, analysis, and tissue culture have been optimized for smaller brain models. This thesis summarizes a body of work aimed at scaling up imaging, analysis, and tissue culture techniques for large-scale brain models. Fluorescence imaging of large-scale models is limited by diffusion of molecular probes and the resolution of long working distance objectives. We present a technique termed SWITCH that inhibits probe binding to allow for diffusion without the formation of a reaction front. To improve imaging resolution, we present a tissue expansion technique called MAP that physically magnifies tissue samples for super-resolution imaging. We demonstrate a 4-fold increase in effective resolution using long working distance objectives and show that the tissue distortion through expansion is comparable to the deformation that occurs during handling. Volumetric imaging of large-scale models generates petabyte scale data, for which we present horizontally scalable image processing pipelines for analysis of intact mouse brains and cerebral organoids. The mouse brain pipeline allows region-based statistical analysis of protein expression and cell counts. The cerebral organoid pipeline allows single-cell, cytoarchitectural, and morphological analyses to be combined into a hyperdimensional statistical analysis. We use this pipeline to show phenotypic changes during development. In order to overcome limitations in multicolor imaging, we also present a 3D image coregistration pipeline capable of aligning multiple rounds of staining of the same whole-brain sample at single-cell resolution. We show that the average distance between corresponding nuclei after coregistration is less than 4 um. Finally, large-scale tissue cultures are limited by nutrient transport since they lack a vascular system. To address this issue in vitro, we fabricated synthetic vasculature by two-photon photopolymerization of polyethylene glycol based resins. Printed micro-vessels were 100 um in outer diameter, durable yet flexible, and permeable to bio-molecules in a tunable manner. Perfusion of vascularized cerebral organoids cultured for 30 days resulted in the expected neuronal differentiation as well as integration of the vascular network. Future studies can use and build on these technical advances to further our understanding of the brain through the use of large-scale brain models.</p>

<h1 id="acknowledgements" class="unnumbered">Acknowledgements</h1>
<p>I would like to acknowledge all of my colleagues at MIT for their continued support throughout various research projects. We accomplished a great deal and learned even more together. I have no doubt that those individuals that I have had the privilege to work with at MIT will continue to impact the world in positive ways.</p>
<p>I am grateful for my fiancé, Kassi, who has been with me side-by-side as an MIT PhD student since the beginning. We made it through grad school together, and I’m excited to share our hardships and victories in the future.</p>
<p>Of course, this work never would have been possible without my family. My brother, Jason, and his wife, Shanon, always gave me a place to stay whenever I went home to Wisconsin during my PhD. My sister, Amy, and neice, Lexi, also welcomed me home when I visited. My dad, Rich, has supported me throughout my entire life and academic career.</p>
<p>Lastly, I’d like to dedicate this thesis to my mother, Margaret Swaney, to whom I owe everything. She has been and will always be my guiding light.</p>




<h1 id="list-of-figures" class="unnumbered">List of figures</h1>
<!--
For me, this was the only drawback of writing in Markdown: it is not possible to add a short caption to figures and tables. This means that the \listoftables and \listoffigures commands will generate lists using the full titles, which is probably isn't what you want. For now, the solution is to create the lists manually, when everything else is finished.
-->
<p>Figure 4.1 This is an example figure . . . <br />
Figure x.x Short title of the figure . . . </p>


<h1 id="list-of-tables" class="unnumbered">List of tables</h1>
<!-- 
For me, this was the only drawback of writing in Markdown: it is not possible to add a short caption to figures and tables. This means that the \listoftables and \listoffigures commands will generate lists using the full titles, which is probably isn't what you want. For now, the solution is to create the lists manually, when everything else is finished.
-->
<p>Table 5.1 This is an example table . . . <br />
Table x.x Short title of the figure . . . </p>
<h1 id="abbreviations" class="unnumbered">Abbreviations</h1>


<h1 id="simple-scalable-proteomic-imaging-for-high-dimensional-profiling-of-intact-systems"><span class="header-section-number">1</span> Simple, Scalable Proteomic Imaging for High-Dimensional Profiling of Intact Systems</h1>
<h2 id="summary"><span class="header-section-number">1.1</span> Summary</h2>
<p>Combined measurement of diverse molecular andanatomical traits that span multiple levels remains amajor challenge in biology. Here, we introduce asimple method that enables proteomic imaging forscalable, integrated, high-dimensional phenotypingof both animal tissues and human clinical samples.This method, termed SWITCH, uniformly securestissue architecture, native biomolecules, and antige-nicity across an entire system by synchronizing thetissue preservation reaction. The heat- and chemi-cal-resistant nature of the resulting framework per-mits multiple rounds (&gt;20) of relabeling. We haveperformed 22 rounds of labeling of a single tissuewith precise co-registration of multiple datasets.Furthermore, SWITCH synchronizes labeling reac-tions to improve probe penetration depth and uni-formity of staining. With SWITCH, we performedcombinatorial protein expression profiling of the hu-man cortex and also interrogated the geometricstructure of the fiber pathways in mouse brains.Such integrated high-dimensional information mayaccelerate our understanding of biological systemsat multiple levels.</p>
<h2 id="introduction"><span class="header-section-number">1.2</span> Introduction</h2>
<p>Biological systems are comprised of vast numbers of molecules,cell types, and intricate tissue organizations (Alivisatos et al.,2013; Kasthuri et al., 2015; Yuste, 2015). Understanding thecomplex interactions of these components is essential formany fields of biology and often requires high-dimensional infor-mation across many scales. Although it is desirable to obtainsuch information from the same tissue due to large individualvariations, combined measurement of many molecular andanatomical traits remains an unmet goal in biology despite theremarkable success of current pioneering methods, such asarray tomography (Micheva et al., 2010; Rah et al., 2013).</p>
<p>Rapidly evolving tissue-clearing techniques may enable multi-plexed labeling and imaging of intact samples using light micro-scopy (Chung et al., 2013; Chung and Deisseroth, 2013; Renieret al., 2014; Richardson and Lichtman, 2015; Susaki et al.,2014). For instance, the CLARITY technique has demonstratedthree rounds of immunostaining of mouse brain tissue (Chunget al., 2013). However, we have noticed that the polyacryl-amide-based framework loses structural integrity upon repeatedexposure to the elution condition. Recent reports also suggestthat preservation of antigenicity in the CLARITY method maynot be optimal (Renier et al., 2014). Furthermore, the necessarytissue-gel hybridization step requires delivery of charged thermalinitiators with limited diffusivity and stability. This necessity im-poses a limit on the tissue size that can be processed withoutthe use of transcardial perfusion.</p>
<p>We set our goal to develop a simple, scalable, and generaliz-able tissue-processing method for proteomic imaging of intactbiological systems. To achieve this, we created SWITCH (sys-tem-wide control ofinteractiontime and kinetics ofchemicals),which tightly controls a broad range of chemical reactions in tis-sue processing via a set of buffers: a SWITCH-On buffer thatfacilitates chemical reactions between exogenous chemicalsand endogenous biomolecules, and a SWITCH-Off buffer thatsuppresses the reactions. SWITCH-mediated fixation trans-forms tissue into a heat- and chemical-resistant hybrid whilepreserving tissue architecture, native molecules, and their anti-genicity to a degree suitable for multiplexed proteomic imaging.The hybrids can be rapidly cleared at high temperature without damage. The method does not require perfusion and is thusapplicable to both animal and large human samples. In molecu-lar labeling of the processed samples, SWITCH controls probe-target binding kinetics to improve probe penetration depth andthe uniformity of molecular labeling. This method is simple, pas-sive, and does not require any special equipment or reagents.</p>
<p>Using SWITCH, we demonstrated that a minimum of 22rounds of molecular labeling of a banked postmortem human tis-sue with precise co-registration of multiple datasets at single-cell resolution is possible. We also demonstrated extraction ofa wide range of system variables, such as various cell typesand microvasculature from a single sample. In summary, wehave developed simple tissue processing methods and a volu-metric co-registration algorithm that can be readily adopted bymost laboratories for scalable proteomic imaging of intact bio-logical systems</p>
<p>To include a citation to the text, just add the citation key shown in the references.bib file. The style of the citation is determined by the ref_format.csl file. For example, in The Living Sea you can find pictures of the Calypso <span class="citation" data-cites="Cousteau1963">(Cousteau Jacques &amp; Dugan James 1963)</span>.</p>
<h2 id="results"><span class="header-section-number">1.3</span> Results</h2>
<h3 id="synchronizing-dialdehyde-tissue-gel-formationenables-scalable-tissue-preservation"><span class="header-section-number">1.3.1</span> Synchronizing Dialdehyde-Tissue-Gel FormationEnables Scalable Tissue Preservation</h3>
<p>First, we sought to develop a way to transform animal and humansamples into a mechanically and chemically stable form for mul-tiplexed imaging. We hypothesized that small, non-ionic, multi-functional crosslinkers might satisfy two key requirements forsuch a transformation: (1) rapid penetration without the use ofperfusion and (2) a high degree of molecular crosslinking toimprove sample durability (Hopwood, 1972; Sung et al., 1996).Among many options, we chose to evaluate the following owingto their small size and high water solubility (Figure 1A): ethyleneglycol diglycidyl ether (EGDGE), dipropylene glycol diglycidylether (GE23), 1,4-butanediol diglycidyl ether (GE21), glycerolpolyglycidyl ether (EX-313), and glutaraldehyde (GA).</p>
<p>We found that all of these chemicals except GE23 formed asolid gel upon incubation with 15% bovine serum albumin(BSA), indicating the formation of a crosslinked network (Fig-ure 1B). We examined the stability of the gels along with poly-acrylamide (AA)-BSA gels by measuring the change in theirvolume after incubation in a 200 mM SDS solution heated to80C (elution condition). AA-BSA gels swelled and became frag-ile after exposure to the harsh condition (Figures 1B and 1C),whereas multi-functional fixative-BSA gels maintained theirstructural integrity. In particular, GA-BSA gels showed minimalvolume change at a wide range of BSA and GA concentrations,whereas others only gelled at high protein concentrations (Fig-ure 1C). This result indicates that multifunctional fixatives alone might be sufficient to form a stable matrix that can withstandthe harsh elution condition. However, because the average pro-tein content throughout mouse brain samples is around 10% andmay be lower within certain regions, we decided that GA is thecrosslinker most likely to form a uniform framework throughoutall regions of a sample.</p>
<p>Next, we asked whether GA can rapidly penetrate tissue toform a uniform tissue-gel without the use of perfusion, which isrequired for processing most human clinical samples. We incu-bated a non-fixed whole adult rat brain in PBS containing 1%GA for 2 days and characterized the GA penetration depth andgel formation. Although the small size of GA should make it highlymobile, only the outer layer of the brain was fixed (Figure 1F).When a coronal slice from the middle of the brain was exposedto the elution condition, the core of the tissue completely disinte-grated, indicating that no gel matrix had formed in the center ofthe brain (Figure 1F). Limited GA penetration has significantlyhampered its use in preserving large postmortem tissues (Hop-wood, 1967). We suspect that rapid reaction of GA with nativebiomolecules within the outer layer of the brain may cause deple-tion of GA molecules before they can reach the core.</p>
<p>To overcome this issue, we sought to control the reaction ki-netics of GA and biomolecules throughout the system usingthe SWITCH approach to achieve uniform tissue preservation.We noted that the GA reaction rate is pH-dependent (Hopwood,1970). Indeed, when we titrated solutions of GA and BSA to pH 3,GA-BSA gel formation time increased by nearly 200-fold (Fig-ure 1D). Using this pH dependence, we were able to disperseGA uniformly throughout a sample by switching off the crosslink-ing reaction with a low-pH buffer (Figure 1E, left). After 2 days ofincubation at low pH, we switched on sample-wide GA-tissuecrosslinking by shifting the pH of the sample to a neutral pH (Fig-ure 1E, right). Using this passive buffer-switching approach, wewere able to achieve complete GA penetration and uniform gelformation throughout the entire rat brain (Figure 1F).</p>
<h3 id="dialdehyde-tissue-gel-preserves-structural-andmolecular-information-effectively"><span class="header-section-number">1.3.2</span> Dialdehyde-Tissue-Gel Preserves Structural andMolecular Information Effectively</h3>
<p>We next asked whether the GA-tissue-gel has mechanical andchemical properties desirable for multiplexing-based proteomicimaging. Proteomic imaging requires (1) high preservation ofendogenous biomolecules and their antigenicity, (2) high struc-tural integrity, and (3) minimal tissue damage during repeatedcycles of destaining, labeling, and imaging processes.</p>
<p>We first tested whether endogenous biomolecules are wellpreserved by measuring protein loss after clearing (see Supplemental Experimental Procedures). We found that controltissues lost an average of 30%–40% protein and AA-tissue-gellost 10%–20%, but GA-tissue-gel slices lost only 3%–5% of theirprotein content (Figure 1G).</p>
<p>We next asked whether antigenicity of the retained biomole-cules is well preserved. We tested 90 antibodies, targeting bio-molecules of different sizes (single amino acid to proteins) andsubcellular localizations (membrane bound, cytoplasm, nucleus,synapses). Surprisingly, 86 of 90 antibodies were compatiblewith GA-tissue-gel (Figures 1H, and 1I, andS1;Table S1). Notethat even small molecules, such as dopamine, which are nottypically compatible with PFA-fixation, were observable in GA-tissue-gel after the complete removal of lipid bilayers (Figure S1).These biomolecules were stable against heat and chemicaltreatment, and their antigenicity was well preserved after expo-sure to elution conditions.</p>
<p>Good structural preservation is essential for resolving proteinlocation with high precision and for studying molecular interrela-tionships. To characterize the macroscale structural preserva-tion of the samples, we cleared 1-mm-thick tissue blocks usingthe elution condition and visualized their structural deformation(Figure 1J). The PFA-only tissue completely disintegrated. Eventhe AA-tissue-gel exhibited large deformations overall. GA-tis-sue-gel, however, showed no signs of structural damagethroughout the entirety of the sample.</p>
<p>We next examined structural preservation on a microscopicscale. We imaged GFP-expressing neurons in the cortex of aPFA-fixed 1-mm-thick thy1-EGFP M line block (Figure 1K). Wethen SWITCH-processed the tissue, cleared it using the harshelution condition, stained it against GFP, and imaged the sameneurons. As shown inFigure 1K, the microscopic morphologyof the neurons was well preserved throughout the entire process.These results show GA-tissue-gel may be ideal for highly multi-plexed structural and molecular phenotyping.</p>
<h3 id="switch-and-robust-computational-algorithms-enablehighly-multiplexed-imaging-at-single-cell-resolution"><span class="header-section-number">1.3.3</span> SWITCH and Robust Computational Algorithms EnableHighly Multiplexed Imaging at Single-Cell Resolution</h3>
<p>Interrogating the three-dimensional (3D) distribution of mole-cules, cells, and the overall tissue organization requires preciseco-registration of multiple volume images. We first asked ifsimple manual overlay of two datasets allows precise co-regis-tration. As a stringent test, we used datasets from multi-roundimaging of a SWITCH-processed 100-mm-thick human brainslice (100mm33,200mm33,200mm) (Figure 2A). The highaspect ratio of such tissues makes it more prone to physicalwarping, which renders co-registration particularly challenging.We first stained the tissue using DAPI and anti-parvalbumin(PV) antibody. The slice was then enclosed in a space larger than the tissue to exaggerate possible tissue deformation inthe mounting process (Figure 2B). After imaging, the samplewas exposed to the elution condition overnight (O/N) tocompletely remove imaged probes. We then restained the tissueusing the same probes and repeated the imaging process. Notethat only GA-tissue-gels could maintain their integrity against theelution treatment. Both AA-tissue-gels and PFA-fixed samplesdeteriorated rapidly in the same condition.</p>
<p>As predicted, a large degree of tissue warping in the mountingprocess (Figure 2C) made manual overlay insufficient for the taskof interrogating a tissue across multiple staining rounds. Toachieve precise co-registration of volume images in the pres-ence of such high-degree warping, we custom-designed arobust computational software based on a feature-detectionapproach that was ideal for our experimental procedure (Fig-ure 2D). Each staining round contained one fluorescence chan-nel devoted to a lectin stain because the morphology of bloodvessels creates distinctive keypoints that computer vision algo-rithms are well suited to identify. With the keypoints, the algo-rithm warps the tissue in a physically plausible manner into thecorrect position (seeSupplemental Experimental Procedures).</p>
<p>As a stringent test of the algorithm, we used the sameSWITCH-processed human sample with the high aspect ratio(Figure 2A). For each round, the sample was stained withDAPI, lectin, and one antibody to label a target protein. Althoughat least three antibodies can be used for each round in additionto lectin and DAPI (Figure S2), we chose to use one antibody foreach round to eliminate any possible cross-talk between chan-nels. After acquiring images, we destained the sample andbegan the next round of labeling. We repeated the above proce-dure 22 times using markers for various cell types (Figure 2H;Ta-ble 1). Staining was not successful in every round due to the useof non-validated antibodies, sub-optimal staining conditions, orhuman error, all of which often occur in general laboratory set-tings and can result in the loss of important samples. However,a SWITCH-processed sample is free from this issue as the tissuecan be washed and reused repeatedly.</p>
<p>We were able to successfully co-register all nine datasets withsuccessful staining (Figures 2E and 2H;Movie S1). We askedwhether changes in the sample might be occurring betweenstaining rounds. To test this, we repeated staining with anti-PVantibodies in rounds 7 and 19 and co-registered the resulting da-tasets. Even when separated by 12 rounds of labeling, we wereable to achieve single-cell accuracy of registration with 99%agreement between the two rounds (Figure 2G).</p>
<p>We next performed joint statistical analysis of the integratedcross-talk-free dataset to extract diverse phenotypic informationfrom human brain (Figure 3). We included lectin, GFAP, NeuN, SMI-32, and three calcium-binding protein channels—calbindin(CB), calretinin (CR), and PV—in the quantitative analysis. First,we used semi-automated algorithms to identify blood vesselsand cells expressing the target antigens (Figures 3A and 3B)and extract their spatial (x, y, z coordinates) and morphological(e.g., cell soma size) information. Density and size profiles ofNeuN-positive cells (Figures 3C and 3D) enabled us to definethe cortical layers (Figure 3A) according to established criteria(de Sousa et al., 2010). NeuN+density was high in cortical layersII and IV, with characteristic small cells (NeuN inFigures 3A, 3C,3D, and 3H). Large NeuN+neurons were concentrated in layersIII and V. A portion of these were large pyramidal neurons posi-tive for SMI-32 (Figures 3A, 3E, and 3H). CB+,CR+, and PV+cellsalso showed distinct distribution patterns along the cortical axis(Figures 3A and 3D), in agreement with previous studies (Defelipeet al., 1999; Leuba et al., 1998).</p>
<p>We next performed unbiased combinatorial expressionprofiling with the six cell-type specific proteins (GFAP, NeuN,SMI-32, CB, CR, PV). Among 63 possible combinations, 16were found (Table S2). We identified sub-populations of CB+/CR+and CB+/PV+cells, but no CR+/PV+or CB+/CR+/PV+, cells(Figures 3F, 3H, and 3I), in agreement with a previous reportregarding mouse visual cortex (Gonchar et al., 2007). Interestingly, we observed that a significant portion of the CB, CR, andPV-positive neurons do not express detectable levels of NeuN,a widely used pan-neuronal marker (Figures 3J and 3K) (Mullenet al., 1992). In particular, a majority of CR+cells showed veryweak (Figure 3K, arrowhead) or no NeuN immunoreactivity(29.1%), whereas all SMI-32+cells (Figures 3J and 3K) wereNeuN positive. These results suggest that NeuN expressionmay be neuronal-type-specific in adult human visual associationcortex. We also found a small number of CB+cells and PV+cellsco-expressing SMI-32, a widely used pyramidal neuronal marker(Table S2) (Campbell and Morrison, 1989). Five CB+/PV+cellswere identified as quadruple-positive (NeuN+/SMI-32+/CB+/PV+) cells (Figure 3G). All of the CB+cells and PV+cells co-ex-pressing SMI-32 were localized in cortical layers III and IV. Theseresults demonstrate the power of SWITCH as a tool for 3D prote-omicprofiling ofintact biological samplesatsingle cellresolution.</p>
<p>Structural relationships between vasculature and brain cellshave been a topic of interest in a broad range of basic and clinicalresearch. Many previous studies obtained the cell-to-vessel dis-tance from 2D images or small tissue volumes, which may hinderprecise measurement of such 3D properties. Moreover, in manystudies, separate measurements from different tissues neededto be compared without considering individual variabilities inlocal vasculature geometry. There has been no direct compari-son of 3D cell-to-vessel distance among diverse cell types withinthe same intact tissue.</p>
<p>Using the proteomic imaging capability of SWITCH, for the firsttime, we were able to directly measure cell-to-vessel distancesfor six different cell types within a single intact tissue (Figures3L–3O). As expected (McCaslin et al., 2011), GFAP+astrocyteshad a shorter mean distance than NeuN+neurons (Figure 3L).CB+and PV+cells were also more closely localized near bloodvessels than NeuN+cells, but the difference was relatively small.Figure 3M shows that vascular density is not uniform along thecortex. However, the extravascular pixel-to-vessel distance(Dp), which we defined as a reference parameter to reflect the ef-fect of the 3D vascular geometry (Figure 3M, right), did not showan inverse relationship with vascular density. This result maysuggest that 3D vessel geometry is an important parameter tobe considered in understanding a given vascular environment.In fact, cell-to-vessel distance profiles of many cell types closelyfollowed theDpprofile (GFAP+,DG,and NeuN+,DN, shown inFigure 3M). In particular, whenDpwas subtracted from cell-to-vessel distances (DX) to cancel the influence of vascular geomet-ric variation,DXDpturns out to be very consistent throughoutcortical depth (Figure 3N). We further examined the distance dis-tribution profiles for all cell types (Figure 3O). All profiles showedsimilar characteristic curves, which can be seen when objectsare randomly located in a 3D space (Manzo et al., 2014). Wecould not observe any cell-type-specific distribution profile orbi- or multi-modal distribution pattern in this sample. Together,these data demonstrate that SWITCH can be used for high-dimensional quantitative phenotyping of human clinical samples.</p>
<h3 id="switch-enables-simple-rapid-and-scalable-tissue-clearing"><span class="header-section-number">1.3.4</span> SWITCH Enables Simple, Rapid, and Scalable Tissue-Clearing</h3>
<p>To extend the multiplexed imaging capability of the SWITCHmethod to large systems, we developed a simple and rapid clearing method. We hypothesized that key steps in detergent-mediated lipid removal, such as permeation of SDS throughmembranes, might be strongly enhanced by increasing temper-ature (Keller et al., 2006), and SWITCH-processed samples mayendure prolonged incubation at elevated temperatures. Indeed,thermal energy drastically increased the passive clearing speedof SWITCH-processed samples without noticeable tissue dam-age (Figure 4A). We achieved passive clearing of a whole adultmouse brain within 4 days at 80C (versus 4 weeks at 37C)(Figure 4C).</p>
<p>Upon prolonged exposure to high temperatures, however,samples developed a brownish hue (Friedman, 1996), whichmay interfere with imaging at certain wavelengths (Figures 4B–4D and 4F). We found that reducing agents, such as sodiumsulfite and 1-thioglycerol, effectively mitigate tissue browningduring thermal clearing (Figures 4B–4D). Using thermal clearingwith the reducing agents, we successfully cleared intact adultrat brains (2 weeks) as well as human (1 week) and marmosetsamples (1 week), demonstrating the versatility and scalabilityof the method (Figures 4D and 4E). Clearing of various rodent or-gans was also demonstrated with lung, kidney, heart, liver, andspinal cord (Figure 4F). The efficacy of sodium sulfite as ananti-browning agent was seen across all tissues.</p>
<h3 id="switch-enables-visualization-and-quantitative-analysisof-entire-myelinated-fiber-tracts"><span class="header-section-number">1.3.5</span> SWITCH Enables Visualization and Quantitative Analysisof Entire Myelinated Fiber Tracts</h3>
<p>We also sought to apply SWITCH to characterizing myelinated fi-ber pathways in the brain. Visualizing and analyzing neural fiberswith high-resolution light microscopy can provide valuable in-sights into many studies (Thomas et al., 2014; Wedeen et al.,2012; Zuccaro and Arlotta, 2013), such as validating diffusiontensor imaging (DTI) and understanding the organizing principlesof brain connectivity. Furthermore, quantitative analysis ofmyelinated fibers in 3D may benefit clinical studies and develop-ment of novel treatments for many demyelinating diseases(Steinman, 1999), such as multiple sclerosis and transversemyelitis. However, current methods for myelinated fiber visuali-zation require either genetic labeling or a large amount of costlyantibodies, limiting their utility to animal tissues or small clinicalsamples (Wedeen et al., 2012).</p>
<p>We discovered that a subset of lipids preserved in SWITCH-processed tissues (Hopwood, 1972; Roozemond, 1969) allowslipophilic dyes to selectively visualize lipid-rich membranes(Schlessinger et al., 1977). In particular, we found that long-chaindialkylcarbocyanines robustly stain myelinated axons (Fig-ure 5A). However, when we attempted to label an intact tissueusing conventional methods, we could not achieve dye penetration deeper than 100mm because dye molecules were depletedas they rapidly associated with abundant targets in the outerlayer (Figure 5C).</p>
<p>We hypothesized that SWITCH may enable rapid and uniformlabeling of intact tissues by synchronizing the labeling reactionglobally. We first screened a range of chemicals for controllingthe binding kinetics of the lipophilic dye and discovered that10 mM SDS effectively inhibits staining (Figure 5B). This result in-dicates that buffers containing 10 mM SDS might have a poten-tial to be used as a ‘‘SWITCH-Off’’ buffer. Using an approachanalogous to SWITCH-mediated GA fixation, we thought it mightbe possible to allow dye molecules to disperse uniformlythroughout a sample in the SWITCH-Off buffer and then activateglobal probe-target binding with the SWITCH-On buffer(Figure 5D).</p>
<p>To test this approach, we first incubated a 1-mm-thick mousebrain block in PBST containing 10 mM SDS and lipophilic dyesfor 24 hr at 37C (SWITCH-Off step). Then, we moved the tissueto PBST and incubated it for 3 hr at 37C (SWITCH-On step). Theresult was strikingly uniform labeling of all the myelinated axonswithin the sample (Figure 5D). Myelinated fibers were clearlyvisible throughout the depth while the control tissue showedsignal only from the surface (Figure 5C).</p>
<p>We leveraged this fiber visualization capability to investigatehow fibers and fascicles are organized in a mouse brain. Previ-ous research has shown that fibers may be organized in 3D grids(Wedeen et al., 2012). However, the structure of all of the individ-ual fibers has not yet been studied at the microscopic resolutionsand macroscopic scales necessary to visualize their 3D organi-zation. To that end, we obtained a volume image of labeledmyelinated fibers in a SWITCH-processed mouse brain coronalslice spanning from the cortex to the striatum (Figure 5E;MoviesS2andS3). This volume shows three main orientations of the fi-bers organized in a cubic grid: one radially projecting from thecorpus callosum and two parallel to the corpus callosum. Thesethree orientations are all orthogonal to one another (Figure 5F;Movie S3). The volume also shows fascicles that radiate fromthe striatum and diverge, almost at right angles, at the corpuscallosum (Figure 5E;Movie S3). To quantify this finding in anon-biased manner, we determined the orientation of each ofthe fibers present in the volume and calculated the angles atwhich these fibers would intersect (Figure 5G). In all three dimen-sions, the fibers indeed oriented themselves approximatelyorthogonally to each other (Figure 5H). We used a similarapproach to examine the fascicle orientations and found thatthey diverge almost orthogonally with respect to the corpus cal-losum in one of the axes (Figure 5I). These results are corroborated by the autocorrelation results (Figure S3andS4).This finding was made possible by the high-resolution andlarge-volume visualization capability of our method. A low-reso-lution approach would overlook the individual fibers while alow-volume approach would be unable to capture the entireconnectional anatomy.</p>
<p>We then tested whether this application of SWITCH could bescaled to larger tissues. We applied the SWITCH approach forlabeling an intact mouse hemisphere, but with 4 days of incu-bation in PBST containing 10 mM SDS and lipophilic dyes(SWITCH-Off step) and 1 day in PBST (SWITCH-On step). Weimaged this larger volume using a custom-built, high-speedlight-sheet microscope (Tomer et al., 2012; Tomer et al.,2014) within 2 hr and observed uniform labeling of all myelin-ated fibers across the entire tissue (Figure 5J;Movie S4). Asdemonstrated, the SWITCH-labeling approach is scalable toorgan-scale tissues. Just by scaling the incubation time withrespect to the tissue size, we were able to label the whole tis-sue. The cost of the dye molecules used for labeling the hemi-sphere was less than one dollar. We also demonstrated thatthis approach can be used for visualizing myelinated fibers inspinal cords (Movie S5). These results show that theSWITCH-labeling method can be used to uniformly label tis-sues ranging from a 1-mm-thick block to an entire hemispherefor quantitative analysis.</p>
<h3 id="switch-enables-scalable-and-uniform-antibody-labeling"><span class="header-section-number">1.3.6</span> SWITCH Enables Scalable and Uniform Antibody Labeling</h3>
<p>We then asked whether SWITCH-mediated labeling could beapplied to the use of antibodies. We hypothesized that SDScould again be used as an effective inhibitor of antibody-antigenbinding in small concentrations. Indeed, when we assayed forantibody labeling at various concentrations of SDS, we foundthat 0.5 to 1.0 mM was a high enough concentration to inhibitbinding for many antibodies (Figure 6A).</p>
<p>Based on the results of our binding assay, we chose PBS con-taining 0.5 mM SDS as a SWITCH-Off buffer and PBST as aSWITCH-On buffer. We hypothesized that, because very littleantibody-antigen binding is occurring in the SWITCH-Off condi-tion, antibodies would effectively be able to diffuse to equilibriumthroughout the sample more rapidly than in PBST, in which anti-bodies are rapidly depleted at the surface (Figure 6B). To testthis, we attempted to label 1-mm-thick mouse brain blocks using anti-histone H3 antibodies. We labeled one sample using a 12 hrSWITCH-Off/12 hr SWITCH-On cycle and another using a stan-dard immunohistochemistry protocol with 12 hr of primary anti-body incubation in PBST followed by a 12 hr wash. For theSWITCH-On step, antibodies were not added to PBST. Theresult was a large increase in penetration depth and overallsignal uniformity in the SWITCH sample relative to the control(Figures 6B and 6C;Movie S6).</p>
<h2 id="discussion"><span class="header-section-number">1.4</span> Discussion</h2>
<p>We have developed SWITCH, a simple method that enablesscalable proteomic imaging of intact systems without requiringany specialized equipment or reagents. SWITCH is complemen-tary to many pioneering technologies, each of which has itsown unique advantages. For example, matrix-assisted laserdesorption ionization mass spectrometry (MALDI-MS) andlaser-ablation inductively coupled plasma mass spectrometry(LA-ICP-MS) allow visualization of a large subset of proteinsand other biomolecules without a priori knowledge of targets.Recent advances in imaging mass spectrometry combinedwith immunohistochemistry (IHC) have significantly improvedresolution (Angelo et al., 2014; Giesen et al., 2014), which waslimited in MALDI-MS and LA-ICP-MS. This approach remarkablydemonstrated analysis of more than 100 targets at subcellularresolution.</p>
<p>Multiplexing strategies for IHC that rely on iterative stainingand elution have been developed. Among several pioneeringtechniques is array tomography, which involves cutting a tissuesample into tens or hundreds of nanometer-thick sections forstaining and imaging (Micheva et al., 2010). These sectionscan be repeatedly washed and stained for probing different pro-teins. This powerful method yields subcellular resolution imagesof a small volume of tissue with fairly high multiplexing capability.Although these advanced technologies enable new approachesin studying complex biological systems, these methods requirespecialized equipment and are, therefore, difficult to implementin most labs.</p>
<p>With the aim of developing a simple and scalable method forproteomic imaging of both large animal and human samples,we first needed to devise the SWITCH method for controlling abroad range of chemical reactions in tissue processing toachieve uniform sample treatment regardless of tissue size and type. SWITCH dynamically modulates chemical reaction kineticsto synchronize the reaction time between molecules throughoutthe system. This strategy enables all endogenous moleculartargets in a large intact tissue to experience similar reaction con-ditions (time and concentration). As a result, large tissues can beuniformly processed.</p>
<p>The SWITCH approach takes advantage of the way certainchemicals can be reversibly and rapidly changed by simplymodulating their surrounding environment. For instance, in theGA-tissue-gelling step, we were able to decrease the rate ofGA-biomolecule crosslinking by two orders of magnitude by us-ing pH 3 buffer, because primary amine groups in endogenousbiomolecules are protonated at low pH and the resulting chargedamine cannot react with GA (Hopwood, 1972). This pH-depen-dent reactivity means that after uniformly dispersing GA in a tis-sue at low pH, we can ‘‘switch-on’’ inactivated amine groups bychanging the amine’s surrounding environment to a neutral-pHbuffer. At neutral pH, charged amine groups are rapidly deproto-nated and become reactive. In the case of human samples or an-imal samples that were previously PFA-fixed for a differentpurpose, this simple strategy enables all the endogenous bio-molecules in a large intact tissue to simultaneously experiencea similar GA-fixation/gelling condition. PFA-fixed tissues canwithstand treatment at low pH while GA molecules are intro-duced. In the case of non-fixed samples, we recommend thatthey first be fixed with PFA before exposure to acidic conditions.If perfusion is possible, it is the recommended method of sample preservation.</p>
<p>Uniform GA-tissue-gel formation is a crucial first step towardour goal. Fixation of large samples via traditional immersion isunlikely to uniformly preserve them because highly reactive GAmolecules are depleted within the outer layers of a sample.This presents a significant problem for iterative staining-basedmethods that rely on the removal of imaged probes using harshelution conditions, because non-uniform preservation results innon-uniform loss of structure and molecules throughout the pro-cess. As demonstrated, our pH-SWITCH strategy ensuresexceptionally uniform preservation of biological tissues thatcannot be perfused (e.g., banked human clinical samples),meeting the requirements of proteomic imaging and quantitative phenotyping.</p>
<p>It has been noted that fixation with GA results in an increase inbroad spectrum autofluorescence. While this autofluorescencehas been low enough to allow quantitative analysis, it could beproblematic in visualizing targets with low copy number. Weinvestigated the use of sodium borohydride as a method ofreducing autofluorescence, but found that the tissue damage resulting from this incubation procedure offset any benefitsobtained from the modest decrease in autofluorescence thatwe were able to observe (Figure S5).</p>
<p>The use of reducing agents has allowed us to eliminate theissue of tissue browning during high-temperature clearing, butwe also observed that excessive use of these chemicals maycause gradual tissue weakening. This is likely due to the reduc-tion of disulfide linkages that maintain the tertiary structure ofproteins within a sample, resulting in increased protein denatur-ation. Protein denaturation may lead to reduced sample antige-nicity, but we have not found this to be an issue when usingconservative amounts of reducing agents. Additionally, due tothe instability of mRNA at elevated temperatures, this methodof rapid clearing is not compatible with methods that requirethe preservation of mRNA (Figure S6).</p>
<p>Multiplexed imaging requires software to warp each experi-ment into a common coordinate system despite the subtle phys-ical differences between each staining round. Variance cancome in the form of rigid body changes (rotation, translation,and scale), illumination artifacts, stain quality, and tissue degra-dation. We observed that a feature-based algorithm givesmaximum robustness across these sources of variance at thecost of increased computational requirements—a reasonabletrade given the declining costs of such resources. To simplifythe process, gross rigid alignments (i.e., rotating the tissue 180degrees) are still best handled by human eye before the data are passed to the algorithm to achieve the cellular-scale registration.</p>
<p>SWITCH can provide a reliable way to obtain integrated high-dimensional information from intact biological samples. Usingthe cross-talk-free dataset, we successfully performednon-biased combinatorial expression analysis of a single hu-man clinical tissue to unequivocally identify diverse cell-typesbased on their distinct protein expression patterns. Our quanti-tative analysis shows that CR+/PV+cells do not exist within theexamined volume of the human V2 cortex. The same findingwas reported in mouse visual cortex (Gonchar et al., 2007),but such co-expression patterns among calcium-bindingproteins may differ among brain regions and between individ-uals and species (Anelli and Heckman, 2005; Defelipe et al.,1999), which, therefore, calls for more comprehensive large-scale investigation.</p>
<p>We observed many NeuN-negative interneurons. NeuN, aneuron-specific RNA-binding protein known as Rbfox3 protein(Kim et al., 2009), has been widely used as a pan-neuronalmarker for statistical analysis of many types of mature neurons(Baleriola et al., 2014; Pickrell et al., 2015). Only a few types ofneurons are exceptions, such as cerebellar Purkinje cells, olfac-tory bulb mitral cells, and retinal photoreceptor cells (Mullenet al., 1992). However, even though we applied strict criteriato prevent weak NeuN+cells from being identified as NeuNcells, substantial portions of CB+,CR+, and PV+neuronswere still NeuNwhile all SMI+neurons were NeuN+. This resultis supported by a recent report that some CR+are not NeuN+,and CR and NeuN immunoreactivities have a negative correla-tion in the avian brainstem (Bloom et al., 2014). Likewise, inour experiments on human visual association cortex, cellswith strong immunoreactivity against calcium-binding protein markers were frequently negative or very weakly positive forNeuN. These findings, together with a series of exceptionalreports such as those on NeuN+cultured astrocytes (Darlingtonet al., 2008) and GFAP+neuron-like cells (Oka et al., 2015), indi-cate that classical cell-type markers, particularly NeuN, mayneed to be used more carefully in light of their selectivity and function.</p>
<p>The SWITCH method has the potential to modulate a widerange of probe-target binding reactions. Probe-target interac-tions are governed by a multiplicity of non-covalent bondssuch as hydrogen bonds, electrostatic forces, van der Waalsbonds, and hydrophobic interactions (Mian et al., 1991). Theseweak forces can be effectively controlled by changing thesurrounding chemical environment (e.g., ionic strength, pH,chemical additive, and temperature) (Kamata et al., 1996).For instance, we discovered that the addition of SDS alone, indifferent concentrations, can completely inhibit lipophilic dye-target and antibody-antigen binding reactions.</p>
<p>The SWITCH method’s unique uniform-labeling capability en-ables quantitative analysis of large tissues that was previouslyonly possible for thin tissue sections. Quantitative analysis reliesheavily on signal intensity and SNR. Non-uniform or heteroge-neous labeling would prohibit or, even worse, bias the analysis.While post hoc image processing methods could correct forsmall gradients in labeling (or imaging), large gradients causedby non-uniform labeling, where the surface of the tissue is satu-rated while the core is mostly unlabeled, would preclude imagerecovery. If the labeling is heterogeneous, the resulting datawould be heavily biased, and no image processing methodscould salvage such data in a fair way. This is why quantitativeanalysis of non-uniformly labeled tissues is a great challenge.However, tissues labeled using SWITCH exhibit uniform signalintensity and SNR throughout the tissue. Such a clear datasetlends itself well to quantitative analysis.</p>
<p>Although SWITCH enables processing of large samples, thespeed of labeling is still fundamentally limited by passive diffu-sion. This is not of concern for smaller samples or even single-round investigation of large samples, but multiplexed imagingof large samples becomes impractical as a result, potentially tak-ing months or years to collect the range of desired data. Recentlydeveloped methods of stochastic electrotransport (Kim et al.,2015) could potentially be combined with SWITCH to facilitatethese experiments.</p>
<p>Together with its simplicity, scalability, and broad applicability,our data suggest that SWITCH provides access to high-dimen-sional multi-scale information that may help to understand healthand disease from molecules to cells to entire systems.</p>
<h2 id="experimental-procedures"><span class="header-section-number">1.5</span> Experimental Procedures</h2>
<h3 id="mice"><span class="header-section-number">1.5.1</span> Mice</h3>
<p>Young adult male and female C57BL/6 and Thy1-eGFP-M mice were housedin a reverse 12 hr light/dark cycle with unrestricted access to food and water.All experimental protocols were approved by the MIT Institutional Animal Careand Use Committee and Division of Comparative Medicine and were in accor-dance with guidelines from the National Institute of NIH.</p>
<h3 id="switch-mediated-tissue-preservation"><span class="header-section-number">1.5.2</span> SWITCH-Mediated Tissue Preservation</h3>
<p>PFA-fixed human samples were washed in a solution consisting of 50% PBStitrated to pH 3 using HCl, 25% 0.1 M HCl, and 25% 0.1 M potassium hydrogenphthalate (KHP). This wash solution was then replaced with fresh solution withthe addition of 4%–10% GA. The samples were then incubated in this pH 3 so-lution at 4C for 2 days with gentle shaking. The solution was then replacedwith PBS with the addition of 1%–4% GA and the sample was again allowedto incubate for 2 days at 4C and 2–7 hr at 37C with gentle shaking. The sam-ple was then washed in PBS at room temperature (RT) for 1 day with gentleshaking. After washing, reactive GA within the sample was inactivated by incu-bation in a solution consisting of 4% glycine and 4% acetamide for 1 day at37C with gentle shaking. Finally, the sample was washed for 1 day in PBS at RT with gentle shaking.</p>
<h3 id="passive-clearing-with-thermal-energy"><span class="header-section-number">1.5.3</span> Passive Clearing with Thermal Energy</h3>
<p>Aqueous clearing solution containing 200 mM SDS, 10 mM lithium hydroxide,40 mM boric acid, and a variable amount of anti-browning agent (i.e., 0–50 mMsodium sulfite or 0%–0.5% [w/v] 1-thioglycerol) was titrated to pH 9 usingsodium hydroxide before use. Samples were incubated at 60–80C until clearusing Easy-Passive (EP-1001; Live Cell Instrument) or a water bath.</p>
<h3 id="sample-delabeling"><span class="header-section-number">1.5.4</span> Sample Delabeling</h3>
<p>Imaged samples were delabeled in clearing solution at 60–80C (elution con-dition) for 1–2 days for large samples and O/N for thin samples.</p>
<h3 id="switch-mediated-fluorescent-labeling"><span class="header-section-number">1.5.5</span> SWITCH-Mediated Fluorescent Labeling</h3>
<p>Samples were incubated in SWITCH-Off solution (0.5 or 10 mM SDS in PBS)O/N with gentle shaking at 37C and transferred to a fresh volume ofSWITCH-Off solution (containing molecular probes) just enough to cover thesample. Samples were incubated at 37C with gentle shaking and timeswere scaled with sample size. Samples were then transferred to a large volumeof PBST (SWITCH-On) and incubated at 37C with gentle shaking.</p>
<h2 id="supplemental-information"><span class="header-section-number">1.6</span> Supplemental Information</h2>
<p>Supplemental Information includes Supplemental Experimental Procedures,six figures, two tables, and six movies and can be found with this article onlineathttp://dx.doi.org/10.1016/j.cell.2015.11.025.</p>
<h2 id="author-contributions"><span class="header-section-number">1.7</span> Author Contributions</h2>
<p>K.C. conceived the SWITCH idea. E.M., J.H.C, D.G., T.K., J.S., and K.C. de-signed the experiments. E.M. designed and performed multiplexed stainingand imaging experiments. D.G. and H.S.S. developed and implemented theregistration algorithm. T.K. performed all analysis related to the human visualcortex sample with input from K.C., M.P.F., and H.S.S. J.H.C. performed allanalysis related to DiD datasets. E.M., N.B., and S.-Y.K. performed antibodycompatibility testing. E.M., J.H.C, N.B., A.H., and J.S. performed testing ofcross-linked protein gels with various fixatives. E.M. performed protein lossand structural preservation experiments. E.M., J.S., and N.B. designed andperformed experiments relating to thermal clearing and use of anti-browningagents. E.M. and M.M. performed DiD staining experiments. E.M. andY.-G.P. performed antibody SWITCH experiments. H.C. designed and builtthe light-sheet microscope, and J.H.C. developed the software. J.-Y.P. pro-vided spinal cord samples. Y.-G.P. performed mRNA experiments. M.P.F. pro-vided human samples. H.S.S., V.J.W., and M.P.F. provided helpful discussionregarding the manuscript. S.V. provided experimental support. E.M., J.H.C.,D.G., T.K., J.S., S.-Y.K., V.J.W., and K.C. wrote the manuscript. K.C. supervised all aspects of the work.</p>
<h2 id="acknowledgments"><span class="header-section-number">1.8</span> Acknowledgments</h2>
<p>We thank the entire Chung laboratory for support and helpful discussions.S.-Y.K. was supported by the Simons Postdoctoral Fellowship and the LifeSciences Research Foundation. K.C. was supported by Burroughs WellcomeFund Career Awards at the Scientific Interface, the Searle Scholars Program, the Michael J. Fox Foundation, DARPA, the JPB Foundation (PIIF and PNDRF),and NIH (1-U01-NS090473-01). M.P.F. was supported in part by the Massa-chusetts Alzheimer Disease Research Center (5 P50 AG005134). Resourcesthat may help enable general users to establish the methodology are freelyavailable online. (www.chunglabresources.org).</p>
<p>Received: August 31, 2015 Revised: October 9, 2015 Accepted: November 10, 2015 Published: December 3, 2015</p>
<h2 id="references"><span class="header-section-number">1.9</span> References</h2>
<p>Alivisatos, A.P., Chun, M., Church, G.M., Deisseroth, K., Donoghue, J.P.,Greenspan, R.J., McEuen, P.L., Roukes, M.L., Sejnowski, T.J., Weiss, P.S.,and Yuste, R. (2013). Neuroscience. The brain activity map. Science339,1284–1285.</p>
<p>Anelli, R., and Heckman, C.J. (2005). The calcium binding proteins calbindin,parvalbumin, and calretinin have specific patterns of expression in the graymatter of cat spinal cord. J. Neurocytol.34, 369–385.</p>
<p>Angelo, M., Bendall, S.C., Finck, R., Hale, M.B., Hitzman, C., Borowsky, A.D.,Levenson, R.M., Lowe, J.B., Liu, S.D., Zhao, S., et al. (2014). Multiplexed ionbeam imaging of human breast tumors. Nat. Med.20, 436–442.</p>
<p>Baleriola, J., Walker, C.A., Jean, Y.Y., Crary, J.F., Troy, C.M., Nagy, P.L., andHengst, U. (2014). Axonally synthesized ATF4 transmits a neurodegenerativesignal across brain regions. Cell158, 1159–1172.</p>
<p>Bloom, S., Williams, A., and MacLeod, K.M. (2014). Heterogeneous calretininexpression in the avian cochlear nucleus angularis. J. Assoc. Res. Otolaryngol.15, 603–620.</p>
<p>Campbell, M.J., and Morrison, J.H. (1989). Monoclonal antibody to neurofila-ment protein (SMI-32) labels a subpopulation of pyramidal neurons in the hu-man and monkey neocortex. J. Comp. Neurol.282, 191–205.</p>
<p>Chung, K., and Deisseroth, K. (2013). CLARITY for mapping the nervous sys-tem. Nat. Methods10, 508–513.</p>
<p>Chung, K., Wallace, J., Kim, S.-Y., Kalyanasundaram, S., Andalman, A.S., Da-vidson, T.J., Mirzabekov, J.J., Zalocusky, K.A., Mattis, J., Denisin, A.K., et al.(2013). Structural and molecular interrogation of intact biological systems.Nature497, 332–337.</p>
<p>Darlington, P.J., Goldman, J.S., Cui, Q.L., Antel, J.P., and Kennedy, T.E.(2008). Widespread immunoreactivity for neuronal nuclei in cultured humanand rodent astrocytes. J. Neurochem.104, 1201–1209.</p>
<p>de Sousa, A.A., Sherwood, C.C., Schleicher, A., Amunts, K., MacLeod, C.E.,Hof, P.R., and Zilles, K. (2010). Comparative cytoarchitectural analyses of stri-ate and extrastriate areas in hominoids. Cereb. Cortex20, 966–981.</p>
<p>Defelipe, J., Gonza ́lez-Albo, M.C., Del Rı ́o,M.R., and Elston, G.N. (1999). Dis-tribution and patterns of connectivity of interneurons containing calbindin, cal-retinin, and parvalbumin in visual areas of the occipital and temporal lobes ofthe macaque monkey. J. Comp. Neurol.412, 515–526.</p>
<p>Friedman, M. (1996). Food Browning and Its Prevention: An Overview. J. Agric.Food Chem.44, 631–653.</p>
<p>Giesen, C., Wang, H.A.O., Schapiro, D., Zivanovic, N., Jacobs, A., Hattendorf,B., Schu ̈ffler, P.J., Grolimund, D., Buhmann, J.M., Brandt, S., et al. (2014).Highly multiplexed imaging of tumor tissues with subcellular resolution bymass cytometry. Nat. Methods11, 417–422.</p>
<p>Gonchar, Y., Wang, Q., and Burkhalter, A. (2007). Multiple distinct subtypes ofGABAergic neurons in mouse visual cortex identified by triple immunostaining.Front. Neuroanat.1,3.</p>
<p>Hopwood, D. (1967). Some aspects of fixation with glutaraldehyde.A biochemical and histochemical comparison of the effects of formaldehydeand glutaraldehyde fixation on various enzymes and glycogen, with a noteon penetration of glutaraldehyde into liver. J. Anat.101, 83–92.</p>
<p>Hopwood, D. (1970). The reactions between formaldehyde, glutaraldehydeand osmium tetroxide, and their fixation effects o bovine serum albumin andon tissue blocks. Histochemie24, 50–64.Cell163, 1500–1514, December 3, 2015ª2015 Elsevier Inc.1513</p>
<p>Hopwood, D. (1972). Theoretical and practical aspects of glutaraldehyde fixa-tion. Histochem. J.4, 267–303.</p>
<p>Kamata, N., Enomoto, A., Ishida, S., Nakamura, K., Kurisaki, J., and Kamino-gawa, S. (1996). Comparison of pH and ionic strength dependence of interac-tions between monoclonal antibodies and bovineb-lactoglobulin. Biosci.Biotechnol. Biochem.60, 25–29.</p>
<p>Kasthuri, N., Hayworth, K.J., Berger, D.R., Schalek, R.L., Conchello, J.A.,Knowles-Barley, S., Lee, D., Va ́zquez-Reina, A., Kaynig, V., Jones, T.R.,et al. (2015). Saturated Reconstruction of a Volume of Neocortex. Cell162,648–661.</p>
<p>Keller, S., Heerklotz, H., and Blume, A. (2006). Monitoring lipid membranetranslocation of sodium dodecyl sulfate by isothermal titration calorimetry.J. Am. Chem. Soc.128, 1279–1286.</p>
<p>Kim, K.K., Adelstein, R.S., and Kawamoto, S. (2009). Identification of neuronalnuclei (NeuN) as Fox-3, a new member of the Fox-1 gene family of splicing fac-tors. J. Biol. Chem.284, 31052–31061.</p>
<p>Kim, S.-Y., Cho, J.H., Murray, E., Bakh, N., Choi, H., Ohn, K., Ruelas, L., Hub-bert, A., McCue, M., Vassallo, S.L., et al. (2015). Stochastic electrotransportselectively enhances the transports of highly electromobile molecules. Proc.Natl. Acad. Sci. USA112, E6274–E6283.</p>
<p>Leuba, G., Kraftsik, R., and Saini, K. (1998). Quantitative distribution of parval-bumin, calretinin, and calbindin D-28k immunoreactive neurons in the visualcortex of normal and Alzheimer cases. Exp. Neurol.152, 278–291.</p>
<p>Manzo, C., van Zanten, T.S., Saha, S., Torreno-Pina, J.A., Mayor, S., and Gar-cia-Parajo, M.F. (2014). PSF decomposition of nanoscopy images viaBayesian analysis unravels distinct molecular organization of the cell mem-brane. Sci. Rep.4, 4354.</p>
<p>McCaslin, A.F.H., Chen, B.R., Radosevich, A.J., Cauli, B., and Hillman, E.M.C.(2011). In vivo 3D morphology of astrocyte-vasculature interactions in the so-matosensory cortex: implications for neurovascular coupling. J. Cereb. BloodFlow Metab.31, 795–806.</p>
<p>Mian, I.S., Bradwell, A.R., and Olson, A.J. (1991). Structure, function and prop-erties of antibody binding sites. J. Mol. Biol.217, 133–151.</p>
<p>Micheva, K.D., Busse, B., Weiler, N.C., O’Rourke, N., and Smith, S.J. (2010).Single-synapse analysis of a diverse synapse population: proteomic imagingmethods and markers. Neuron68, 639–653.</p>
<p>Mullen, R.J., Buck, C.R., and Smith, A.M. (1992). NeuN, a neuronal specific nu-clear protein in vertebrates. Development116, 201–211.</p>
<p>Oka, Y., Ye, M., and Zuker, C.S. (2015). Thirst driving and suppressing signalsencoded by distinct neural populations in the brain. Nature520, 349–352.</p>
<p>Pickrell, A.M., Huang, C.-H., Kennedy, S.R., Ordureau, A., Sideris, D.P., Hoekstra, J.G., Harper, J.W., and Youle, R.J. (2015). Endogenous Parkin PreservesDopaminergic Substantia Nigral Neurons following Mitochondrial DNA Muta-genic Stress. Neuron87, 371–381.</p>
<p>Rah, J.-C., Bas, E., Colonell, J., Mishchenko, Y., Karsh, B., Fetter, R.D., Myers,E.W., Chklovskii, D.B., Svoboda, K., Harris, T.D., and Isaac, J.T. (2013). Tha-lamocortical input onto layer 5 pyramidal neurons measured using quantitativelarge-scale array tomography. Front. Neural Circuits7, 177.</p>
<p>Renier, N., Wu, Z., Simon, D.J., Yang, J., Ariel, P., and Tessier-lavigne, M.(2014). iDISCO: A Simple, Rapid Method to Immunolabel Large Tissue Sam-ples for Volume Imaging. Cell159, 896–910.</p>
<p>Richardson, D.S., and Lichtman, J.W. (2015). Clarifying Tissue Clearing. Cell162, 246–257.</p>
<p>Roozemond, R.C. (1969). The effect of fixation with formaldehyde and glutar-aldehyde on the composition of phospholipids extractable from rat hypothal-amus. J. Histochem. Cytochem.17, 482–486.</p>
<p>Schlessinger, J., Axelrod, D., Koppel, D.E., Webb, W.W., and Elson, E.L.(1977). Lateral transport of a lipid probe and labeled proteins on a cell mem-brane. Science195, 307–309.</p>
<p>Steinman, L. (1999). Assessment of animal models for MS and demyelinatingdisease in the design of rational therapy. Neuron24, 511–514.</p>
<p>Sung, H.W., Hsu, H.L., Shih, C.C., and Lin, D.S. (1996). Cross-linking charac-teristics of biological tissues fixed with monofunctional or multifunctionalepoxy compounds. Biomaterials17, 1405–1410.</p>
<p>Susaki, E.A., Tainaka, K., Perrin, D., Kishino, F., Tawara, T., Watanabe, T.M.,Yokoyama, C., Onoe, H., Eguchi, M., Yamaguchi, S., et al. (2014). Whole-brainimaging with single-cell resolution using chemical cocktails and computationalanalysis. Cell157, 726–739.</p>
<p>Thomas, C., Ye, F.Q., Irfanoglu, M.O., Modi, P., Saleem, K.S., Leopold, D.A.,and Pierpaoli, C. (2014). Anatomical accuracy of brain connections derived from diffusion MRI tractography is inherently limited. Proc. Natl. Acad. Sci.USA111, 16574–16579.</p>
<p>Tomer, R., Khairy, K., Amat, F., and Keller, P.J. (2012). Quantitative high-speedimaging of entire developing embryos with simultaneous multiview light-sheetmicroscopy. Nat. Methods9, 755–763.</p>
<p>Tomer, R., Ye, L., Hsueh, B., and Deisseroth, K. (2014). Advanced CLARITY forrapid and high-resolution imaging of intact tissues. Nat. Protoc.9, 1682–1697.</p>
<p>Wedeen, V.J., Rosene, D.L., Wang, R., Dai, G., Mortazavi, F., Hagmann, P.,Kaas, J.H., and Tseng, W.-Y.I. (2012). The geometric structure of the brain fiberpathways. Science335, 1628–1634.</p>
<p>Yuste, R. (2015). From the neuron doctrine to neural networks. Nat. Rev. Neu-rosci.16, 487–497.</p>
<p>Zuccaro, E., and Arlotta, P. (2013). The quest for myelin in the adult brain. Nat.Cell Biol.15, 572–575.1514Cell163, 1500–1514, December 3, 2015ª2015 Elsevier Inc.</p>
<h1 id="ultrafast-immunostaining-of-organ-scale-tissues-for-scalable-proteomic-phenotyping"><span class="header-section-number">2</span> Ultrafast immunostaining of organ-scale tissues for scalable proteomic phenotyping</h1>
<h2 id="abstract-1"><span class="header-section-number">2.1</span> Abstract</h2>
<p>Studying the function and dysfunction of complex biological systems necessitates comprehensive understanding of individual cells. Advancements in three-dimensional (3D) tissue processing and imaging modalities have enabled rapid visualization and phenotyping of cells in their spatial context. However, system-wide interrogation of individual cells within large intact tissue remains challenging, low throughput, and error-prone owing to the lack of robust labeling technologies. Here we introduce a rapid, versatile, and scalable method, eFLASH, that enables complete and uniform labeling of organ-scale tissue within one day. eFLASH dynamically modulates chemical transport and reaction kinetics to establish system-wide uniform labeling conditions throughout the day-long labeling period. This unique approach enables the same protocol to be compatible with a wide range of tissue types and probes, enabling combinatorial molecular phenotyping across different organs and species. We applied eFLASH to generate quantitative maps of various cell types in mouse brains. We also demonstrated multidimensional cell profiling in a marmoset brain block. We envision that eFLASH will spur holistic phenotyping of emerging animal models and disease models to help assess their functions and dysfunctions.</p>
<h2 id="introduction-1"><span class="header-section-number">2.2</span> Introduction</h2>
<p>System-wide analysis of cell types is essential for understanding how complex cellular interactions give rise to various functions. Extensive efforts have been made towards characterizing cells, particularly in the brain, through various lenses (e.g., genomics, transcriptomics, proteomics, connectomics) and have established invaluable databases with new insights 1–7 . Among these approaches, proteomic imaging has distinct advantages. Mapping spatial distribution of proteins, the major functional substrate with distinct subcellular localization at single molecule precision, can provide rich molecular, functional, as well as morphological details of cells. Furthermore, visualizing endogenous proteins with highly specific antibodies does not require genetic manipulation or invasive in vivo surgery, and thus it is applicable to any species or tissue type including non-human primates and human clinical samples 8 .</p>
<p>When combined with intact organ transformation and clearing techniques, proteomic phenotyping can provide multiscale information, ranging from brain-wide cell distribution patterns to molecular and morphological details of individual cells without information loss caused by subsampling or 2D analysis 9–11 . However, scaling immunolabeling to large-scale tissues and higher species remains a major challenge in biology. Passive transport of large macromolecules such as antibodies into intact tissues can take weeks to months 9,12 . Antibody penetration can be further delayed or even blocked by target proteins with high expression levels, causing probe depletion and incomplete staining. Using excessive amounts of antibodies can improve probe penetration, but it becomes prohibitively expensive and thus unscalable. In conventional passive labeling approaches, experimental parameters for labeling (e.g., incubation time, probe amount) are highly dependent on sample properties (e.g., tissue type, size, shape) and target protein properties (e.g., expression level, distribution patterns), which are widely different between applications. Therefore, each experiment requires laborious, costly, and time-consuming optimization. The outcome of passive labeling is in many cases highly uneven with saturated labeling of outer regions and weak or no labeling of the core. Such non-uniform and incomplete labeling can prohibit automated analysis and cause systematic error. These challenges together have limited the power of 3D proteomic phenotyping to small tissues or a small number of established applications.</p>
<p>Here we present an integrated pipeline for holistic, rapid, scalable proteomic phenotyping of intact organs. To establish the pipeline, we developed an ultrafast and versatile immunolabeling technology, termed eFLASH (electrophoretically driven Fast Labeling using Affinity Sweeping in Hydrogel), which enables complete and uniform labeling of various types of tissues (mouse brain and intestine, human iPSC-derived cerebral organoid, and marmoset brain block) using a wide selection of antibodies (targeting structural, molecular, and neuronal activity markers) with a universal 1-day protocol. Combined with intact tissue processing and analysis techniques, we performed organ-wide quantification of various proteins at cellular resolution in mouse brains. We further demonstrated the power of 3D protein-based cell phenotyping by characterizing neural sub-types based on their 3D location, protein expression level, cell body size, and dendritic morphology in a fully integrated manner.</p>
<h2 id="results-1"><span class="header-section-number">2.3</span> Results</h2>
<h3 id="eflash-mechanism"><span class="header-section-number">2.3.1</span> eFLASH mechanism</h3>
<p>Our organ-wide molecular phenotyping framework consists of four major components: (1) intact tissue preservation via SHIELD, (2) volumetric labeling with eFLASH, (3) light-sheet imaging, and (4) automated 3D image analysis (Fig. 1a). The pipeline begins with robust preservation of biological tissue with SHIELD, which is a polyepoxide-based tissue fixation method that protects biomolecules and tissue architecture 13 . After rapid delipidation of the SHIELD-preserved tissues using stochastic electrotransport (SE) 12 , we immunolabel the intact tissues using eFLASH within just one day. The labeled samples are rapidly imaged at high-resolution using an axially swept light-sheet microscope. Finally, we analyze the resulting volumetric datasets via a suite of automated 3D image analysis algorithms to map various cell types within the tissue volume. Altogether, the pipeline enables extraction of organ-scale, single-cell-resolution data from a fresh sample within just 12 days (Fig. 1a).</p>
<p>eFLASH allows uniform immunolabeling of organ-scale tissues within a day by gradually shifting probe-target binding conditions from unfavorable to favorable while accelerating probe penetration using stochastic electrotransport (Fig. 1b). We discovered that bile salts, such as sodium deoxycholate (NaDC), can be used to control the labeling affinity for various antibodies in a concentration and pH dependent manner (Fig. 1c-d; Supplementary Fig. 1). A wide range of probes showed weak binding at high concentrations of NaDC and basic pH, but strong binding at low concentration of NaDC and neutral pH. These results indicate that labeling conditions can be gradually shifted from unfavorable to favorable by simultaneously sweeping pH (basic to neutral) and NaDC concentration (high to low).</p>
<p>To achieve a gradual pH sweep in an automated manner, we took advantage of electrochemical reactions that naturally occur under SE. Electrocatalytic oxidation of D-sorbitol produces acidic byproducts such as formic acid 14 . By adding D-sorbitol to a pH 9.5 buffer and letting it decompose by electro-oxidation under SE, we were able to gradually sweep pH from 9.5 to 7.5 over the course of one day (Fig. 1b, Supplementary Fig. 2).</p>
<p>Concentration of NaDC within the labeling solution was also swept in an automated manner using the concentration gradient established across a nanoporous membrane (Fig. 1b). The membrane, which separates the labeling solution and the outer solution, ensures that both molecular probes and large NaDC micelles remain within the labeling solution; however, it is permeable to NaDC monomers, small NaDC aggregates, and the rest of the buffer components. The initial concentration of 1% (w/v) NaDC within the labeling solution slowly decreases as the monomers travel down the concentration gradient to the outer solution, which contains 0.2% (w/v) of NaDC (Fig. 1b). We confirmed that the terminal buffer after pH and NaDC concentration shift allows strong antibody staining (Fig. 1c-d).</p>
<p>This progressive change in binding condition enables the probes to first penetrate deep into the tissue without being depleted and then increasingly bind to targets globally as the buffer composition gradually changes. This approach ensures uniform labeling of entire volumes regardless of the density and distribution pattern of the targets, specific binding kinetics of various antibodies, and the amount of antibody used. With eFLASH, even when using a minute amount of antibody for labeling highly dense targets (3 g of antibody for calbindin and 5 g of antibody for pan-axonal marker SMI312), high-quality uniform labeling could be achieved in a mouse brain hemisphere (Fig. 1e); however, without affinity-sweep, the small amount of antibody was quickly depleted on the surface before the core of the tissue could be labeled despite the increased transport speed provided by SE (Fig. 1f, Supplementary Video 1). These results indicate that eFLASH enables rapid, complete, and uniform immunolabeling of organ-scale tissues without the use of excessive amounts of molecular probes.</p>
<h3 id="universal-applicability-of-eflash"><span class="header-section-number">2.3.2</span> Universal applicability of eFLASH</h3>
<p>The affinity sweeping mechanism in eFLASH renders the technique insensitive to tissue type, size, or geometry. eFLASH is also insensitive to probe types because the sweeping range is wide enough to modulate binding affinities of many antibodies and other commonly used molecular probes. Therefore, the same operational parameters of eFLASH can be used for many applications without laborious and costly optimization. We found that a single protocol with the same parameters (e.g., voltage, pH, running time, chemical concentrations) is capable of uniformly labeling cerebral organoid, mouse intestine, mouse brain hemisphere, as well as marmoset brain block with various combinations of antibodies, allowing visualization of multiple proteins within a single sample (Fig 2a-d, Supplementary video 2-3).</p>
<p>The same 1-day protocol was compatible with a wide range of antibodies harboring different binding affinities and target densities (Fig. 2e). eFLASH successfully labeled targets for various cell types (PV, CB, CR, NPY, SST, TH, TPH2, ChAT, NeuN, GFAP, Iba1), neuronal structure (SMI-312), and a neuronal activity marker (cFos) in intact mouse hemispheres (Fig. 2e, Supplementary video 4-5). The same eFLASH protocol was also compatible with lectin and Syto16, which are chemical probes that label blood vessels and nuclei, respectively. Together, these results suggest that eFLASH is a universal platform compatible with a wide range of tissue- types and molecular probes.</p>
<h3 id="a-quantitative-brain-wide-cell-type-mapping-with-eflash"><span class="header-section-number">2.3.3</span> A quantitative, brain-wide cell type mapping with eFLASH</h3>
<p>eFLASH, combined with light-sheet microscopy, enables true volumetric quantification of protein expression at cellular resolution. eFLASH-stained mouse brain hemispheres were rapidly imaged using an axially swept light-sheet microscope at near-isometric resolution of 1.8 m x 1.8 m x 2 m within 45 minutes per channel. Because the sample was processed and imaged as a whole without sectioning, the resulting volumetric data is an exhaustive representation of the sample that does not suffer from sampling errors and does not require interpolation or extrapolation to acquire brain-wide or region-specific cell counts. In addition, the multiplexed labeling capability of eFLASH allows analysis of cells co-expressing multiple proteins of interest with relative ease and flexibility compared to genetic labeling approaches. Currently, labeling up to four distinct targets is possible through transgenic labeling approaches 15 ; however, developing transgenic mouse lines for each new combination of targets can be time consuming 16 .</p>
<p>To demonstrate the value of holistic labeling with eFLASH, we established an image analysis pipeline for atlas alignment, brain region segmentation, and cell detection for generating a quantitative map of various proteins. Volumetric images were automatically aligned to an atlas 4 by linear and non-linear transformations based on Elastix 17 then manually refined 18 . Each aligned 3D image volume was indexed to approximately 580 brain regions with 7 hierarchies. Brain-wide quantification of immunolabeled cells was accomplished using machine learning algorithms that were trained to identify features of individual cell-types (Supplementary Fig. 3). Specifically, Random Forest 19 was applied after blob detection and principal component analysis (PCA). Detection performance was validated against manual ground-truth annotations of relevant brain regions that are known to express each cell type. Our cell detection pipeline achieved an f-score of higher than 90% for cortical regions and over 80% for subcortical brain regions for all tested cell-type markers. Using this pipeline, we were able to construct quantitative mouse brain atlases for multiple cell type defining makers, including CR, NPY, SST, TH, TPH2, and PV (Fig. 3a-c, Supplementary video 6).</p>
<p>The probe-insensitive nature of eFLASH enables co-labeling of multiple cell-types with any combinations. We performed simultaneous labeling of neuropeptide Y and somatostatin which are known to be co-expressed in a subset of GABAergic interneurons 20–22 and of Tyrosine Hydroxylase and Tryptophan Hydroxylase 2 which are cell-type-specific markers for dopaminergic and serotonergic neurons, respectively, that are not generally known to overlap. In the case of NPY and SST, we mapped NPY+/SST-, NPY-/SST+, and NPY+/SST+ cells (Fig. 3 d- f). We found the highest density of NPY+ cells at layer 2 or 3 of the cerebral cortices (Fig. 3c), whereas SST+ cells showed the highest density at layer 4 or 5 in majority of the cortices 23 (Fig. 3c) Interestingly, the highest density of cells that were co-positive for NPY and SST was seen in layer 5 or 6 (Fig. 3f). In a brain-wide average, 16 ± 4 % of NPY and 7 ± 5% of SST-expressing cortical cells were identified NPY+ / SST+ co-positive. In the case of TH and TPH2, we checked every TH+ and TPH2+ cells detected throughout the brain hemisphere and found that no cells were positive for both markers.</p>
<p>Finally, in addition to labeling cell-type defining proteins, brain-wide labeling of Immediate Early Genes (IEGs) such as c-Fos has been demonstrated as a powerful proxy for measuring neuronal activation 24,25 . We stained the brain of a mouse that experienced contextual fear conditioning 90 minutes before sacrifice with anti-cFos antibody and mapped its distribution (Fig. 3g-i). Examination of the dataset showed robust anti-c-Fos signal in hippocampus and amygdala areas, which are known to show increased activity upon contextual fear conditioning 26 . Combined, these results suggest that eFLASH-mediated immunolabeling can facilitate brain-wide quantification of protein expression at a cellular level in a high throughput and flexible manner.</p>
<h3 id="brain-wide-comparison-of-genetic-and-protein-based-cell-type-labeling"><span class="header-section-number">2.3.4</span> Brain-wide comparison of genetic and protein-based cell type labeling</h3>
<p>Expression of genetically encoded fluorescent proteins have revolutionized biological labeling and imaging 27 , and ongoing developments in transgenic methodology offer powerful ways to study organ-wide gene expression 28–30 . However, the level of fluorescent protein expression is linked to transcription activity rather than the level of expression of mRNA or proteins, requiring careful and nuanced interpretation of data 31,32 . Additionally, several studies have reported on the importance of post-transcriptional processes that can often cause the quantities of mRNA and proteins to correlate poorly 32 , emphasizing the need for protein expression analysis.</p>
<p>Discrepancy between transgenic labeling and immunohistochemical labeling is widely recognized 28 , and there is a constant concerted effort to improve upon existing transgenic mouse lines for common targets 33–35 . To compare genetic labeling and eFLASH-mediated cell-type phenotyping approaches, we utilized transgenic mouse lines with two widely used transgene approaches: Cre-LoxP and BAC transgene 28,36–38 . First, we eFLASH-stained a hemisphere of a PV-Cre::DIO-tdTomato double transgenic mouse with anti-PV antibody (Fig. 4a). We performed the brain-wide quantitative analysis on tdTomato and anti-PV signals, and revealed substantial discrepancies between two labeling approaches, where the level of mismatch was highly heterogeneous among brain regions (Fig. 4b-c, Supplementary video 7). For example, in contrast to faithful tdTomato labeling of PV+ neurons in primary motor and primary somatosensory cortices (88% and 85% of tdTomato+ cells were also PV+), a substantial portion of tdTomato cells showed undetectable amounts of PV protein in some of cortical (e.g., 56% and 75% in the case of piriform and lateral entorhinal cortex) and subcortical (45% in caudate putamen; 62% in nucleus accumbens) areas. Furthermore, we found PV+ populations were not covered by the genetic labeling. For example, 66% and 77% of PV+ cells do not express tdTomato in CPu and NAc, respectively (Fig. 4b-c).</p>
<p>Next, we compared genetic and protein-based labeling of choline acetyltransferase (ChAT). eGFP expression via BAC transgene was highly divergent from the ChAT+ immunoreactivity pattern (Fig. 4d-h, Supplementary video 8). For example, only 9% and 14% of eGFP+ cells were also ChAT+ in M1 and S1 cortex. In the hippocampal CA1 and CA3, only 0.2% and 0.3% of eGFP+ cells showed detectable levels of ChAT immunoreactivity. Further, large populations of ChAT+ cells without eGFP expression were evident, especially in primary auditory and visual cortices (93%, 89%) (Fig. 4f). These discrepancies were heterogeneous even within the same brain region. Most ChAT+ cells were also eGFP+ in Nucleus ambiguus ventral part (80%), however, in its dorsal counterpart, only 26% of ChAT+ cells were colocalized with eGFP+ (Fig. 4e-iii). 3D visualization of the hemisphere also revealed labeling mismatch between fiber bundles. In the brain stem, we found a fiber bundle composed of ChAT+ axons without eGFP signals (Fig. 4g,h). These results suggest that eFLASH enables brain-wide analysis of transgenic labeling patterns and their validation by allowing simultaneous visualization of genetically expressed fluorescent proteins and immunolabeling signal within the same sample.</p>
<h3 id="multidimensional-single-cell-analysis-of-marmoset-visual-cortex"><span class="header-section-number">2.3.5</span> Multidimensional single-cell analysis of marmoset visual cortex</h3>
<p>Common marmoset (Callithrix jacchus), a small New World primate, has emerged as a powerful model for neuroscience research 39 . Their rapid reproduction cycles and compatibility with existing genetic engineering tools renders them a promising model for studying various brain disorders. Holistic cell-level phenotyping of the marmoset brain, however, remains challenging owing to the limited quality and availability of transgenic lines, significantly higher cost and larger brain size compared to rodent models.</p>
<p>Protein-based cellular phenotyping using eFLASH and SHIELD can be advantageous for higher model systems, including primates, where genetic manipulation remains challenging 40,41 . Moreover, the multiplexing capability of this approach allows simultaneous mapping of various molecular and cell-type markers within the same brain tissue, which not only increases the dimensionality of integrated phenotypic analysis, but also decreases the number of animals required for a study and consequently the cost.</p>
<p>To test this idea, we applied eFLASH and SHIELD to characterize cells in an intact marmoset brain block of visual cortex (5 mm x 5 mm x 8 mm). First, we eFLASH-stained the SHIELD-preserved sample with anti-PV antibody. From the holistic visualization and detection of PV+ cells in the sample (Fig. 5a), we found that the inter-layer distribution of PV+ cell is heterogeneous among parts of the visual cortex. The density of PV+ cells was higher in the area facing the calcarine sulcus compared to the other area of the visual cortical block (1770.3 ± 56.4 vs 979.0 ± 33.4 cells per mm 3 , unpaired T-test, P&lt;0.0005, N= 4 of 120 m-thick optical sections) (Fig. 5b-c, Supplementary video 9). We also observed that several cortical areas were devoid of PV+ neurons (Fig. 5a-c). Furthermore, we observed that inter-layer distribution patterns of PV+ cells differed between mouse and marmoset visual cortex (Fig. 5c). After mapping PV+ cells, we destained the same marmoset brain block and re-stained it with anti-NPY antibody using eFLASH. We found that NPY+ cells are mostly localized in layer 6 and white matter of the marmoset visual cortex, which was in contrast with mouse visual cortex that showed a more uniform NPY+ cell distribution across the cortical layers (Fig. 5d-f).</p>
<p>Immunostaining can provide access to cellular morphology without genetic labeling or dye injections because many proteins are distributed or transported to cytoplasm and subcellular compartments. Using eFLASH-mediated volume-wide immunolabeling, we may be able to characterize both morphological and molecular details of individual cells throughout intact tissue volumes. To demonstrate this possibility, we performed deep analysis of individual NPY+ cells in a cortical fold sub-volume. From the automatically detected 6796 NPY+ cells in the volume, we quantified the soma volume and mean immunointensity of 494 cells, and dendrite polarity of 119 cells (Fig. 5g-i). Analysis of dendritic morphology of individual NPY+ cells led us to classify the cells into separate categories based on previously established descriptions of GABAergic interneurons: bitufted, bipolar or multipolar 42 (Fig 5h, Supplementary Fig 4.) Compared to NPY+ cells in white matter, NPY+ cells in gray matter have soma with larger volume and higher mean fluorescent signal intensity (Fig. 5k-l), suggesting higher intracellular concentration of NPY protein 43,44 . We also found that most gray matter NPY+ cells are multipolar cells, whereas most NPY+ cells in white matter were bitufted or bipolar cells (Fig. 5h,j,m; Supplementary video 10). Together, these results demonstrate that complete and uniform immunolabeling of large-scale intact tissues with eFLASH enables high-dimensional phenotyping of individual cells even on model animals with limited access to genetic tools.</p>
<h2 id="discussion-1"><span class="header-section-number">2.4</span> Discussion</h2>
<p>In this study, we developed a rapid, versatile, and scalable immunolabeling technology, eFLASH, that enables complete and uniform immunolabeling of organ-scale tissues within one day for protein-based high dimensional cellular phenotyping. The universal 1-day protocol based on the gradual sweeping of probe-target binding affinity allows labeling of various markers simultaneously for disparate tissue types. Combined with the volumetric imaging and analysis pipeline, eFLASH enables 3D visualization and multi-dimensional phenotyping of molecular markers in large intact tissues with single-cell-resolution.</p>
<p>eFLASH is rationally designed to address the main challenge in scaling molecular labeling to organ-scale samples: the drastic mismatch between probe diffusion time scale and probe- target reaction time scale. Probe-target binding reaction is orders of magnitude faster than probe diffusion 12 . The diffusion timescale increases quadratically with the thickness of the sample, whereas probes rapidly bind to targets as soon as they encounter. If the density of the target molecule is high, which is the case for many of protein targets, probes cannot penetrate deeper into the tissue until they saturate all target molecules in their path. This means that uniform and complete labeling of intact tissue is not possible without using a large amount of probes, reducing the tissue size, or reducing the density of antigens.</p>
<p>Transport of electromobile molecules such as antibodies can be expedited using stochastic electrotransport 12 . However, the probe transport time scale in SE is still much longer than the reaction time scale. Applying a higher electric field can further increase transport speed, but Joule heating can cause tissue damage. Therefore, it is imperative to modulate both the rate of reaction and transport simultaneously. Switching off the binding reaction allows transport of antibodies into the core of the tissue without depletion 45 . Once probes reach the core of the sample, the binding reaction can be switched back on by changing the surrounding chemical environment (pH, detergent concentration). Discrete modulation of kinetics by such step-wise change, however, inevitably forms concentration gradients of chemicals (e.g., pH and NaDC) and probes inside the tissue, which causes uneven labeling. We addressed this challenge in eFLASH by slowly and gradually changing the concentration of the chemicals to ensure that the reaction condition is uniform tissue-wide throughout the day-long labeling period.</p>
<p>eFLASH is a robust process that offers considerable experimental flexibility. Repeated staining of the same tissue is possible with eFLASH, allowing multiple interrogations of precious samples as demonstrated with the marmoset brain block (Fig. 5). eFLASH can also be used to immunolabel the organs of transgenic mice expressing fluorescent proteins, allowing simultaneous visualization of both genetic labeling and immunolabeling signals (Fig. 2c, Fig. 4). This suggests that eFLASH can be utilized for comprehensive immunohistological validation of genetic labeling, amplification of genetically labeled signal using anti-fluorescent protein antibodies, and multiplexed proteomic analysis of genetically labeled cells in intact tissues.</p>
<p>Recently, tissue-clearing techniques and volume imaging methods have been applied to whole organ samples to demonstrate the potential of 3D phenotyping with single-cell resolution 24,46,47 . Many of these studies utilized genetic labeling which provides both uniform and high signal-to-noise ratio suitable for computational analysis 29 . However, genetic labeling is relatively inflexible when it comes to target selection, as new transgenic mouse or protocol is required for each target or each combination of targets 16,36 . With eFLASH, the choice of targets and the combinations of targets is based simply on the availability of compatible molecular probes. Additionally, eFLASH performs direct immunohistological labeling of target proteins present in the tissue, allowing for simplified interpretation of resulting data. As powerful as it is, Cre-LoxP transgenesis is known to suffer from false-positive (e.g., transgene-independent CRE expression, CRE-independent recombination) and false-negative labeling (e.g., CRE mosaicism) 48 . We observed that there was a discrepancy between fluorescent protein signal and antibody labeling signal in PV-Cre:DIO-tdTomato double-transgenic mouse brain labeled with anti-PV antibody (Fig. 4), Reversible tissue and temporal specific control systems, (such as a tetracycline response system), and BAC transgenesis resolved some of these issues, but not all. For example, discrepancies between genetic and protein-based labeling in BAC transgenic mouse lines were observed in both previous 28,34,49 and present studies (Fig. 4), and it has been suggested that expression of BAC transgene can be affected by the presence of other transcription factors, microRNAs, or control regions of gene fragments 16,50 . We anticipate that protein-based mapping enabled by eFLASH can complement the cutting-edge genetic labeling approaches (e.g., viral labeling) for anatomical, molecular, and functional mapping of neural circuits.</p>
<p>Furthermore, eFLASH can facilitate studies of animal models with limited access to genetic labeling methodologies. The Common Marmoset is an emerging primate model for social behaviors with many experimental advantages 51,52 , and thus much effort has been undertaken to construct marmoset brain atlases in diverse modalities 53–55 . Unfortunately, numerous hurdles remain in translating existing genetic labeling approaches for rodents to marmosets. For example, germline genetic manipulation for generating transgenic primates is still difficult and expensive 40,56 , and long gestation and maturation period of primates as well as ethical concerns make each primate sample highly precious. Viral labeling approaches have shown the most promise; however, clear limitations exist since most enhancer elements are not defined and viral vectors have limited capacity to include large gene elements 40 . Moreover, achieving systemic coverage of the entire brain with viral labeling also remains challenging 57 . We anticipate that the scalability and flexibility of eFLASH will aid organ-wide phenotyping efforts on such model animals.</p>
<p>We envision that the versatility and high throughput capabilities of eFLASH will benefit numerous studies requiring system-wide yet highly detailed views of biological tissues, especially for exploratory studies comparing healthy and diseased animals or of model animals with limited access to genetic labeling strategies. Application of eFLASH will synergize greatly with advancements in biological imaging, molecular binder technologies, and computational frameworks for big data analysis 58 . Holistic, rapid, and unbiased approaches enabled by such synergistic technological advances will ultimately aid in providing a broader perspective in the study of complex biological systems.</p>
<h2 id="acknowledgements-1"><span class="header-section-number">2.5</span> Acknowledgements</h2>
<p>The authors thank the entire Chung laboratory for support and discussions. We acknowledge N. Peat for contribution to chemical screening for buffer development. K.C. was supported by the Burroughs Wellcome Fund Career Awards at the Scientific Interface, Searle Scholars Program. Packard award in Science and Engineering, NARSAD Young Investigator Award, and the McKight Foundation Technology Award. This work was supported by the JPB Foundation (PIIF and PNDRF), the NCSOFT Cultural Foundation, the Institute for Basic Science IBS-R026-D1, and the NIH (1-DP2-ES027992, U01MH117072).</p>
<h2 id="author-contributions-1"><span class="header-section-number">2.6</span> Author Contributions</h2>
<p>D.H.Y., Y.-G.P., J.H.C. and K.C. designed the experiments and wrote the paper with input from other authors. D.H.Y., J.H.C., and K.C. designed eFLASH protocols and systems. D.H.Y. and J.H.C. performed the volumetric labeling experiments with N.D.’s help. Y.-G.P. aided the development of the eFLASH technology by performing passive staining experiments for screening antibodies and buffers, and imaging eFLASH-labeled samples. Y.-G.P. led SHIELD-processing of all tissue samples with K.X.’s help. G.F. and K.C. initiated the marmoset brain mapping project. G.F. provided the marmoset and Q.Z. perfused the marmoset. L.K. and J.S. developed the computational pipeline with Y.-G.P., D.H.Y., W.G., and K.C.’s input. N.B.E. and Y.-G.P. performed light-sheet imaging with H.C.’s help. D.H.Y. performed active delipidation of mouse and marmoset samples with N.D.’s help. D.H.Y. performed the buffer characterization in Figure 1. A.A. provided and imaged the SHIELD processed cerebral organoid for Figure 2. Y.-G.P. and L.K. performed brain-wide cell-type mapping in Figure 3 with D.H.Y. and K.X.’s help. Y.-G.P. performed co- positivity analysis for Figure 4 and the multi-dimensional analysis of marmoset datasets in Figure 5. C.H.S. aided in antibody and fluorescent dye screening for the project. G.D. and Y.X. helped with initial manuscript preparation. Y.X., H.-Y.J., and L.R. aided in detergent and buffer screening and characterization.</p>
<h2 id="competing-interests"><span class="header-section-number">2.7</span> Competing Interests</h2>
<p>K.C. and D.H.Y. are co-inventors on a patent application owned by MIT covering the eFLASH technology. K.C. and J.H.C. are co-inventors on patents owned by MIT covering the SWITCH and SE technology.</p>
<h2 id="references-1"><span class="header-section-number">2.8</span> References</h2>
<p>1.Sharma, K. et al.Cell type-and brain region-resolved mouse brain proteome. Nat. Neurosci.18, 1819–1831 (2015).</p>
<p>2.Hawrylycz, M. J. et al.An anatomically comprehensive atlas of the adult human brain transcriptome. Nature489, 391–399 (2012).</p>
<p>3.Miller, J. A. et al.Transcriptional landscape of the prenatal human brain. Nature508, 199–206 (2014).</p>
<p>4.Oh, S. W. et al.A mesoscale connectome of the mouse brain. Nature508, 207–214 (2014).</p>
<p>5.Wang, X. et al.Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Science361, eaat5691 (2018).</p>
<p>6.Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. &amp; Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. Science (80-. ).348, 1360–1363 (2015).</p>
<p>7.Habib, N.et al.Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Science353, 925–8 (2016).</p>
<p>8.Belle, M. et al.Tridimensional Visualization and Analysis of Early Human Development. Cell169, 161-173.e12 (2017).</p>
<p>9.Chung, K. &amp; Deisseroth, K. CLARITY for mapping the nervous system. Nat. Methods10, 508–513 (2013).</p>
<p>10.Renier, N. et al.IDISCO: A simple, rapid method to immunolabel large tissue samples for volume imaging. Cell159, 896–910 (2014).</p>
<p>11.Luo, L., Callaway, E. M. &amp; Svoboda, K. Genetic Dissection of Neural Circuits: A Decade of Progress. Neuron98, 256–281 (2018).</p>
<p>12.Kim, S.-Y. et al.Stochastic electrotransport selectively enhances the transport of highly electromobile molecules. Proc. Natl. Acad. Sci.112, E6274–E6283 (2015).</p>
<p>13.Park, Y.-G. G. et al.Protection of tissue physicochemical properties using polyfunctional crosslinkers. Nat. Biotechnol.37, 73 (2019).</p>
<p>14.Proença, L. et al.Electrocatalytic oxidation of D-sorbitol on platinum in acid medium: Analysis of the reaction products. J. Electroanal. Chem.432, 237–242 (1997).</p>
<p>15.Gaire, J. et al.PrismPlus: a mouse line expressing distinct fluorophores in four different brain cell types. Sci. Rep.8, 7182 (2018).</p>
<p>16.Liu, C. Strategies for designing transgenic DNA constructs. Methods Mol. Biol.1027, 183–201 (2013).</p>
<p>17.Klein, S., Staring, M., Murphy, K., Viergever, M. A. &amp; Pluim, J. P. W. elastix: A Toolbox for Intensity-Based Medical Image Registration. Med. Imaging, IEEE Trans.29, 196–205 (2010).</p>
<p>18.Swaney, J. et al.Scalable image processing techniques for quantitative analysis of volumetric biological images from light-sheet microscopy. bioRxiv576595 (2019). doi:10.1101/576595</p>
<p>19.Breiman, L. Random Forests. Mach. Learn.45, 5–32 (2001).</p>
<p>20.Fu, L.-Y. &amp; van den Pol, A. N. GABA excitation in mouse hilar neuropeptide Y neurons. J. Physiol.579, 445–64 (2007).</p>
<p>21.Figueredo-CardenaS, G., Morello, M., Sancesario, G., Bernardi, G. &amp; Reiner, A. Colocalization of somatostatin, neuropeptide Y, neuronal nitric oxide synthase and NADPH-diaphorase in striatal interneurons in rats. Brain Res.735, 317–324 (1996).</p>
<p>22.Hendry, S. H., Jones, E. G. &amp; Emson, P. C. Morphology, distribution, and synaptic relations of somatostatin-and neuropeptide Y-immunoreactive neurons in rat and monkey neocortex. J. Neurosci.4, 2497–517 (1984).</p>
<p>23.Rudy, B., Fishell, G., Lee, S. &amp; Hjerling-Leffler, J. Three groups of interneurons account for nearly 100% of neocortical GABAergic neurons. Dev. Neurobiol.71, 45–61 (2011).</p>
<p>24.Renier, N. et al.Mapping of Brain Activity by Automated Volume Analysis of Immediate Early Genes. Cell165, 1789–1802 (2016).</p>
<p>25.Kim, Y. et al.Mapping social behavior-induced brain activation at cellular resolution in the mouse. Cell Rep.10, 292–305 (2015).</p>
<p>26.Cho, J.-H., Rendall, S. D. &amp; Gray, J. M. Brain-wide maps of Fos expression during fear learning and recall. Learn. Mem.24, 169–181 (2017).</p>
<p>27.Giepmans, B. N. G., Adams, S. R., Ellisman, M. H. &amp; Tsien, R. Y. The fluorescent toolbox for assessing protein location and function. Science312, 217–24 (2006).</p>
<p>28.Gong, S. et al.A gene expression atlas of the central nervous system based on bacterial artificial chromosomes. Nature425, 917–925 (2003).</p>
<p>29.Kim, Y. et al. Brain-wide Maps Reveal Stereotyped Cell-Type-Based Cortical Architecture and Subcortical Sexual Dimorphism. Cell171, 456-469.e22 (2017).</p>
<p>30.Zhang, C. et al.A platform for stereological quantitative analysis of the brain-wide distribution of type-specific neurons. Sci. Rep.7, 14334 (2017).</p>
<p>31.Huang, Z. J., Taniguchi, H., He, M. &amp; Kuhlman, S. Genetic labeling of neurons in mouse brain. Cold Spring Harb. Protoc.2014, 150–60 (2014).</p>
<p>32.Vogel, C. &amp; Marcotte, E. M. Insights into the regulation of protein abundance from proteomic and transcriptomic analyses. Nat. Rev. Genet.(2012). doi:10.1038/nrg3185</p>
<p>33.Tallini, Y. N. et al.BAC transgenic mice express enhanced green fluorescent protein in central and peripheral cholinergic neurons. Physiol. Genomics27, 391–397 (2006).</p>
<p>34.Li, X. et al.Generation of a whole-brain atlas for the cholinergic system and mesoscopic projectome analysis of basal forebrain cholinergic neurons. Proc. Natl. Acad. Sci. U. S. A.115, 415–420 (2018).</p>
<p>35.Madisen, L. et al.A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Nat. Neurosci.13, 133–140 (2010).</p>
<p>36.Muzumdar, M. D., Tasic, B., Miyamichi, K., Li, L. &amp; Luo, L. A global double-fluorescent Cre reporter mouse. genesis45, 593–605 (2007).</p>
<p>37.Livet, J. et al.Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system. Nature450, 56–62 (2007).</p>
<p>38.Valjent, E., Bertran-Gonzalez, J., Hervé, D., Fisone, G. &amp; Girault, J.-A. Looking BAC at striatal signaling: cell-specific analysis in new transgenic mice. Trends Neurosci.32, 538–547 (2009).</p>
<p>39.Miller, C. T. et al.Marmosets: A Neuroscientific Model of Human Social Behavior. Neuron90, 219–33 (2016).</p>
<p>40.He, M. &amp; Huang, Z. J. Genetic approaches to access cell types in mammalian nervous systems. Curr. Opin. Neurobiol.50, 109–118 (2018).</p>
<p>41.Belmonte, J. C. I. et al.Brains, Genes, and Primates. Neuron87, 671 (2015).</p>
<p>42.(PING), T. P. I. N. G. et al.Petilla terminology: nomenclature of features of GABAergic interneurons of the cerebral cortex. Nat. Rev. Neurosci.9, 557–568 (2008).</p>
<p>43.Chung, K. et al.A microfluidic array for large-scale ordering and orientation of embryos. Nat. Methods8, 171–176 (2011).</p>
<p>44.Watanabe, J., Asaka, Y. &amp; Kanamura, S. Relationship between immunostaining intensity and antigen content in sections. J. Histochem. Cytochem.44, 1451–8 (1996).</p>
<p>45.Murray, E. et al.Simple, Scalable Proteomic Imaging for High-Dimensional Profiling of 16Intact Systems. Cell163, 1500–1514 (2015).</p>
<p>46.Murakami, T. C. et al.A three-dimensional single-cell-resolution whole-brain atlas using CUBIC-X expansion microscopy and tissue clearing. Nat. Neurosci.21, 625–637 (2018).</p>
<p>47.Yang, B. et al.Single-cell phenotyping within transparent intact tissue through whole-body clearing. Cell158, 945–958 (2014).</p>
<p>48.Heffner, C. S. et al.Supporting conditional mouse mutagenesis with a comprehensive cre characterization resource. Nat. Commun.3, 1218 (2012).</p>
<p>49.von Engelhardt, J., Eliava, M., Meyer, A. H., Rozov, A. &amp; Monyer, H. Functional Characterization of Intrinsic Cholinergic Interneurons in the Cortex. J. Neurosci.27, 5633–5642 (2007).</p>
<p>50.Matthaei, K. I. Genetically manipulated mice: a powerful tool with unsuspected caveats. J. Physiol.582, 481–8 (2007).</p>
<p>51.Mitra, P. Brain-mapping projects using the common marmoset. Neurosci. Res.93, 3–7 (2015).</p>
<p>52.Kaiser, T. &amp; Feng, G. Modeling psychiatric disorders for developing effective treatments. Nat. Med.21, 979–988 (2015).</p>
<p>53.Lin, M. K. et al.A high-throughput neurohistological pipeline for brain-wide mesoscale connectivity mapping of the common marmoset. Elife8, 1–36 (2019).</p>
<p>54.Woodward, A. et al.The Brain/MINDS 3D digital marmoset brain atlas. Sci. Data5, 180009 (2018).</p>
<p>55.Liu, C. et al.A digital 3D atlas of the marmoset brain based on multi-modal MRI. Neuroimage169, 106–116 (2018).</p>
<p>56.Jennings, C. G. et al.Opportunities and challenges in modeling human brain disorders in transgenic primates. Nat. Neurosci.19, 1123–1130 (2016).</p>
<p>57.Matsuzaki, Y. et al.Intravenous administration of the adeno-associated virus-PHP.B capsid fails to upregulate transduction efficiency in the marmoset brain. Neurosci. Lett.665, 182–188 (2018).</p>
<p>58.Fürth, D. et al.An interactive framework for whole-brain maps at cellular resolution. Nat. Neurosci.21, 139–149 (2018).</p>
<h2 id="methods"><span class="header-section-number">2.9</span> Methods</h2>
<h3 id="mice-1"><span class="header-section-number">2.9.1</span> Mice</h3>
<p>Young adult (2-4 month) C57BL/6 mice were housed in a 12 hr light/dark cycle with unrestricted access to food and water. All experimental protocols were approved by the MIT Institutional Animal Care and Use Committee and the Division of Comparative Medicine and were in accordance with guidelines from the National Institute of Health. The following transgenic lines were used for this study: Thy1::GFP M-line, Thy1::YFP H-line, ChAT BAC -eGFP (Jackson Stock No. 007902), PV-Cre / loxP-tdTomato (Jackson Stock No. 017320 and 007914), and Fos-CreER T2 / DIO-tdTomato (Jackson Stock No. 021882, 007914).</p>
<h3 id="marmoset"><span class="header-section-number">2.9.2</span> Marmoset</h3>
<p>All animal experiments were approved by the Institutional Animal Care and Use Committee of Massachusetts Institute of Technology and were performed under the guidelines from the National Institute of Health. Adult common marmosets (2-4 years old) were housed in AAALAC-accredited facilities. The housing room was maintained at 74.0 ± 2.0 °F (23.3 ± 1.1 °C), in the relative humidity of 50 ± 20 %, and in a 12 hr light/dark cycle. The animals were housed in dedicated cages with enrichment devices and had unrestricted access to food and water.</p>
<p>For histological examinations, the animals were deeply sedated by intramuscular injection of Ketamine (20-40 mg/kg) or Alfaxalone (5-10 mg/kg), followed by intravenous injection of sodium pentobarbital (10-30 mg/kg). When pedal withdrawal reflex was eliminated and/or respiratory rate was diminished, animals were perfused transcardially with 0.5 ml 1000 IU/ml heparin and 100-200 ml cold PBS by gravity. Then the descending aorta of the animals was clamped, and a peristaltic pump was used to infuse another 200-300 ml ice-cold SHIELD perfusion solution (10%(w/v) GE38 and 4% PFA(w/v) in PBS). Brains were removed from the skulls and SHIELD-processed.</p>
<h3 id="organoids"><span class="header-section-number">2.9.3</span> Organoids</h3>
<p>Organoids were grown according to the protocol by Lancaster et al. 59 , with the addition of dual SMAD inhibition between d6 and d9 to increase neural differentiation as previously described 60 . Organoids were grown from iPSC cells (System Biosciences, #SC101A- 1). After Matrigel droplet embedding, organoids were transferred to 60 mm suspension culture dishes (Corning, #430589) and placed on shaker at 75 rpm on day 16. The organoids were SHIELD-processed at day 35 (see the section “SHIELD processing”).</p>
<h3 id="contextual-fear-conditioning"><span class="header-section-number">2.9.4</span> Contextual fear conditioning</h3>
<p>Contextual fear conditioning (CFC) was conducted using a chamber with an animal shocker (Habitest, Coulbourn, MA). After 300 s exploration in the chamber, mice were shocked (0.75 mA, 2 s) and maintained in the chamber 5 minutes more. Mice were sacrificed 60 minutes after the behavioral test was ended.</p>
<h3 id="sodium-deoxycholate-nadc-concentration-measurement"><span class="header-section-number">2.9.5</span> Sodium Deoxycholate (NaDC) Concentration Measurement</h3>
<p>Concentration of surfactants can be measured by the degree of solubilization of hydrophobic organic dyes. Above the critical micelle concentration, the amount of solubilized dye increases linearly with the increase in surfactant concentration 61 . Degree of solubilization was measured based on light absorption using a microplate reader at 505nm. Sufficient Orange OT dye (Sigma, 344664, powder) was added to fully saturate 200-proof ethanol at RT. 200 l of saturated solution was added to each of the wells in 96-well plate and allowed to fully evaporate to deposit Orange OT dye to the well surface. 100 l of eFLASH buffer collected at various time points were added to the prepared wells and left on an orbital shaker overnight. The well plate was centrifuged at 2000g for 10 minutes (Multifuge X1R, Thermofisher). 50 l from each well was collected and added to a black 96-well plate with glass bottom for measurement using a microplate reader (EnSpire Multimode Plate Reader, Perkinelmer). NaDC concentration was calculated based on a standard curve generated using the method described above from solutions with known concentrations of NaDC (Supplementary Figure 2).</p>
<h3 id="shield-processing"><span class="header-section-number">2.9.6</span> SHIELD Processing</h3>
<p>Preservation of mouse brain hemispheres were carried out according to the previously published SHIELD protocol 3 . Mice were transcardially perfused with ice-cold PBS and then with the SHIELD perfusion solution. Dissected brains or organs were incubated in the same perfusion solution at 4 °C for 48 h. Tissues were then transferred to the SHIELD-OFF solution (1X PBS containing 10% (w/v) P3PE) and incubated at 4 °C for 24 h. In the case of brain hemisphere processing, a whole brain was split into hemispheres before being incubated in the SHIELD-OFF solution. Following the SHIELD-OFF step, the organs were placed in the SHIELD-ON solution (0.1 M sodium carbonate buffer at pH 10) and incubated at 37 °C for 24 h.</p>
<p>Marmoset brains perfused with ice-cold PBS and then with SHIELD perfusion solution were incubated in the same perfusion solution at 4 °C for 48 h. The brain was hemisected, transferred to the SHIELD-OFF solution, and incubated at 4 °C for 24 h. Following the SHIELD-OFF step, the hemispheres were placed in the SHIELD-ON solution and incubated at 37 °C for 24 h. Afterwards the hemispheres were transferred to PBS for washing.</p>
<p>Organoids were fixed in 1X PBS with 4% (w/v) PFA at RT for 30 minutes and subsequently incubated in SHIELD-OFF solution at 4°C for 48h. Samples were then incubated in SHIELD-ON solution at 37°C overnight before washing with PBS with 0.02% sodium azide at RT for at least 24 h.</p>
<h3 id="passive-clearing-delipidation"><span class="header-section-number">2.9.7</span> Passive Clearing (Delipidation)</h3>
<p>SHIELD-processed samples were delipidated before labeling or imaging. Passive delipidation was done by incubating tissues in the clearing buffer (300 mM SDS, 10 mM sodium borate, 100mM sodium sulfite, pH 9.0). Thin slices between 100 m and 200 m thickness were cleared at 45°C clearing buffer for 2-3 hrs. Mouse brain hemispheres were cleared at 45°C for 10-14 days. Organoids were cleared at 55°C for 36 hrs.</p>
<h3 id="active-clearing-stochastic-electrotransport"><span class="header-section-number">2.9.8</span> Active Clearing (Stochastic Electrotransport)</h3>
<p>SHIELD-processed samples can also be cleared rapidly using stochastic electrotransport (SmartClear Pro, LifeCanvas Technologies). Mouse brain hemispheres were cleared at 45°C for 3-4 days. The marmoset brain hemisphere was cut coronally into 4 blocks of 8 mm-thickness using a microtome and the blocks were cleared at 45°C for 2 weeks.</p>
<h3 id="antibody-delabeling"><span class="header-section-number">2.9.9</span> Antibody Delabeling</h3>
<p>Imaged SHIELD tissue was first equilibrated with the clearing buffer (200mM SDS, pH 9.5) at 37°C overnight. Afterwards the sample was moved to a separate falcon tube with 50 mL of clearing buffer that was preheated to 80°C and kept on a heated shaker maintained at 80°C for 1 h. Afterwards, the solution was exchanged with fresh clearing buffer at RT and the sample was incubated on an orbital shaker at 37°C overnight. The sample was washed using PBS with multiple solution exchanges for one day to thoroughly wash out SDS.</p>
<h3 id="passive-immunohistochemistry"><span class="header-section-number">2.9.10</span> Passive Immunohistochemistry</h3>
<p>Immunohistochemistry was performed on 100 m- or 200 m-thick mouse or marmoset brain tissue sections. Staining was performed on 24 or 48 well plates with primary antibodies (per recommended dilution from each vendors) and with dye- conjugated Fc-specific Fab fragments (3:1 molar ratio between Fab fragments and the primary antibody, Jackson Immunoresearch) for 1 day at RT in PBS with 0.1% Triton-X100. Similar protocols were used to characterize antibody binding performance in several different buffers: PBS with 0.1% NaDC, PBS with 1% NaDC, eFLASH initial buffer (240mM Tris, 160mM CAPS, 20% w/v D-sorbitol, 0.9% w/v NaDC, pH 9.6), and eFLASH terminal buffer (buffer retrieved from the eFLASH staining device after 24 h, pH 7.4).</p>
<h3 id="eflash-protocol"><span class="header-section-number">2.9.11</span> eFLASH Protocol</h3>
<p>Volumetric immunolabeling with eFLASH was carried out with a device described in Kim et al. 12 Experiments were carried out with two buffers. The main buffer (240mM Tris, 160mM CAPS, 20% (w/v) D-sorbitol, 0.2% (w/v) NaDC) is a circulation solution that allows conduction of electricity. The sample buffer (240mM Tris, 160mM CAPS, 20% (w/v) D-sorbitol, 0.9% (w/v) NaDC) is used to fill the the sample cup along with the tissue and antibodies. 300 mL of a booster buffer (20% w/v D-sorbitol, 60mM Boric Acid) was added to the main buffer at 20 h after the start of the experiment to achieve the desired pH in the sample cup at 24 h.</p>
<p>300-500 mL of the main buffer was loaded into the staining device and 2-5 mL of the sample buffer was loaded into the sample cup. The tissue sample was placed in a nylon mesh then placed into the sample cup. Primary antibodies (antibody information and optimized quantity for each target, Supplementary Table 1) and secondary antibodies were added to the sample cup. Dye-conjugated Fc-specific Fab fragments were used for all experiments (2:1 molar ratio to the primary antibody, Jackson immunoresearch). The machine was operated for 24 h at 90 V with maximum current limited to 500 mA. Temperature control was set to maintain 25°C. Sample cup stir bar rotation was set to 850 rpm and sample cup rotation speed was set to 0.01 rpm.</p>
<h3 id="dye-conjugation-of-secondary-antibodies"><span class="header-section-number">2.9.12</span> Dye Conjugation of Secondary Antibodies</h3>
<p>For the far-red channel, secondary antibodies conjugated with SeTau647 were used for most labeling experiments as they provide superior photo-stability when compared to commercially available dyes 62 . SeTau-647-NHS was purchased from SETA BioMedicals and 10 l 10mM aliquots were prepared using DMSO (anhydrous, ZerO2®, ≥99.9%, Sigma). SeTau-647-NHS were reacted with non-conjugated Fc- specific Fab fragments at 10:1 ratio (Jackson Immunoresearch) for 1 h at RT. Afterwards, the solution was purified using Zeba Spin Desalting Columns (7k MWCO, ThermoFisher Scientific) 2 to 3 times until the desalting column ran clean. The concentration of the resulting solution was measured using DC TM Protein Assay (Bio-Rad) before use.</p>
<h3 id="refractive-index-matching"><span class="header-section-number">2.9.13</span> Refractive Index Matching</h3>
<p>Optical clearing of delipidated samples was achieved using Protos-based immersion medium 13 . For samples thicker than 1 mm, optical clearing was done in a step-wise manner. Labeled samples were first incubated in half-step solution (50/50 mix of 2X PBS and Protos-based immersion medium) at 37°C overnight. Afterwards, the samples were moved to the pure immersion medium and incubated at 37°C overnight.</p>
<h3 id="fixation-of-labeled-samples"><span class="header-section-number">2.9.14</span> Fixation of Labeled Samples</h3>
<p>For antibodies that are not stable in Protos-based immersion medium, the eFLASH-labeled samples were fixed with 4% (w/v) PFA to prevent dissociation of bound antibodies. eFLASH-labeled samples in the terminal labeling buffer were washed in 1X PBS with 0.02% (w/v) sodium azide at RT for at least 6 h to wash out Tris in the sample. Samples were then moved to freshly prepared 4% (w/v) PFA solution in 1X PBS, and placed on an orbital shaker at RT overnight. Samples were then washed with 1X PBS with 0.02% (w/v) sodium azide at RT with multiple solution exchanges for at least 6 h.</p>
<h3 id="light-sheet-imaging-and-post-processing"><span class="header-section-number">2.9.15</span> Light-sheet Imaging and Post-processing</h3>
<p>Rapid volumetric imaging was performed with an axially swept light-sheet microscope (SmartSPIM, Lifecanvas Technologies, MA) equipped with three lasers (488nm, 561nm, 642nm). The scanning was fine-tuned for each sample by finely adjusting the position of the illumination objectives to ensure optimal optical sectioning. Focus compensation was programmed as a function of depth for each laser line to account for slight focal variations through imaging depth. All light-sheet imaging was done with either of the following objective lenses: 3.6x objective (custom Lifecanvas design, 0.2NA 12mm WD lateral resolution 1.8um in XY), 10x objective (Olympus XLPLN10XSVMP, 0.6NA, 8mm WD, lateral resolution 0.66um in XY). Acquired data was post-processed with algorithms described in Swaney et al. 18 . A complete table of imaging modalities and conditions for every data included in this paper can be found in Supplementary table 2.</p>
<h3 id="cell-detection"><span class="header-section-number">2.9.16</span> Cell Detection</h3>
<p>Detection of cells is accomplished by blob detection, followed by dimensionality reduction and classification. Blobs are detected by computing the difference of Gaussians followed by identification of voxels that are the maximum of their neighbors within a chosen radius. 31x31 pixel patches are then extracted in the X/Y, X/Z and Y/Z planes. The rasters of these patches are concatenated and the three resulting 961-element vectors are concatenated to create a 2883-feature vector. All patches of putative cell centers within the volume are collected and PCA is performed to reduce the dimensionality of the vector to 48 components. Each of these components are composed of 2883 elements which are multiplied with the 2883- feature vector per patch to produce 48 numerical features. The vector of each component can be visualized as three 31x31 planes (see Supplementary Figure 3) to allow interpretation of the magnitude of the component. The 48 numerical features are then used to train a random forest classifier using iterative user-supervised training. Finally, the classifier is applied to all patches in the volume to classify each local maximum as a positive cell detection or negative artifact detection.</p>
<h3 id="atlas-alignment"><span class="header-section-number">2.9.17</span> Atlas Alignment</h3>
<p>Atlas alignments of mouse brain hemispheres labeled with eFLASH to the Allen brain reference atlas, CCF V3 63 , were carried out using the hybrid automated atlas alignment method described in Swaney et al 18 , which combines Elastix 17 and manual refinement tools to improve alignment accuracy.</p>
<h3 id="brain-region-segmenation"><span class="header-section-number">2.9.18</span> Brain Region Segmenation</h3>
<p>Detected cell coordinates were transformed from the original coordinate space to the reference coordinate after atlas alignment. The alignment was used to construct a three-dimensional radial basis function using thin-plate spines to map points in the original coordinate space to the reference coordinate space. The point locations in the reference space were then matched against the Allen Brain Mouse Atlas reference 64 segmentation to yield counts per brain region. These counts were then used to color the regions in the Allen Brain Mouse Atlas coronal SVG image files. Calculations and visualizations were done using the Nuggt python package 18 .</p>
<h3 id="manual-image-analysis"><span class="header-section-number">2.9.19</span> Manual Image Analysis</h3>
<p>Imaris (Bitplane, Switzerland) was used for soma segmentation, analysis, and neurite tracing in figure 5g-m. Dendrite polarity of NPY+ cells were assessed manually 65 . Fluorescence quantification was done using ImageJ.</p>
<h3 id="code-availability"><span class="header-section-number">2.9.20</span> Code Availability</h3>
<p>The custom code used in this study is available from the corresponding author upon reasonable request.</p>
<h3 id="data-availability"><span class="header-section-number">2.9.21</span> Data Availability</h3>
<p>The data supporting the findings of this study are available from the corresponding author upon reasonable request.</p>
<p>59.Lancaster, M. A. &amp; Knoblich, J. A. Generation of cerebral organoids from human pluripotent stem cells. Nat. Protoc.9, 2329–2340 (2014).</p>
<p>60.Mellios, N. et al.MeCP2-regulated miRNAs control early human neurogenesis through differential effects on ERK and AKT signaling. Mol. Psychiatry 2017 23423, 1051 (2017).</p>
<p>61.Tehrani-Bagha, A. R. &amp; Holmberg, K. Solubilization of Hydrophobic Dyes in Surfactant Solutions. Materials (Basel).6, 580 (2013).</p>
<p>62.Podgorski, K., Terpetschnig, E., Klochko, O. P., Obukhova, O. M. &amp; Haas, K. Ultra-Bright and -Stable Red and Near-Infrared Squaraine Fluorophores for In Vivo Two-Photon Imaging. PLoS One7, e51980 (2012).</p>
<p>63.Dong, H. The Allen reference atlas: A digital color brain atlas of the C57Bl/6J male mouse.(2008).</p>
<p>64.Lein, E. S. et al.Genome-wide atlas of gene expression in the adult mouse brain. Nature445, 168–176 (2007).</p>
<p>65.Nela, P. &amp; Vera, T. LOCALISATION AND MORPHOLOGY OF COCAINE-AND AMPHETAMINE-REGULATED TRANSCRIPT (CART) PEPTIDE IMMUNOREACTIVE NEURONS IN RAT AMYGDALA. Acta Veterinaria (Beograd)55, (2005</p>
<h1 id="multiplexed-and-scalable-super-resolution-imaging-of-three-dimensional-protein-localization-in-size-adjustable-tissues"><span class="header-section-number">3</span> Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues</h1>
<h2 id="summary-1"><span class="header-section-number">3.1</span> Summary</h2>
<p>The biology of multicellular organisms is coordinated across multiple size scales, from the subnanoscale of molecules to the macroscale, tissue-wide interconnectivity of cell populations. Here we introduce a method for super-resolution imaging of the multiscale organization of intact tissues. The method, called magnified analysis of the proteome (MAP), linearly expands entire organs fourfold while preserving their overall architecture and three-dimensional proteome organization. MAP is based on the observation that preventing crosslinking within and between endogenous proteins during hydrogel-tissue hybridization allows for natural expansion upon protein denaturation and dissociation. The expanded tissue preserves its protein content, its fine subcellular details, and its organ-scale intercellular connectivity. We use off-the-shelf antibodies for multiple rounds of immunolabeling and imaging of a tissue’s magnified proteome, and our experiments demonstrate a success rate of 82% (100/122 antibodies tested). We show that specimen size can be reversibly modulated to image both inter-regional connections and fine synaptic architectures in the mouse brain.</p>
<h2 id="introduction-2"><span class="header-section-number">3.2</span> Introduction</h2>
<p>Biological systems such as the mammalian brain consist of thou- sands of distinct cell types forming highly interconnected func- tional networks 1–5 . Understanding how these diverse cells interact to generate system-level responses is essential for many fields of biology. Deciphering these complex interactions requires tools that can simultaneously characterize the molecular identity and fine sub- cellular architectures of individual cells as well as their system-level connectivity, because these properties together determine many cell functions. To date, technologies provide only a subset of the required multilevel information.</p>
<p>Proteomic imaging techniques can potentially provide multilevel information in both animals and human samples 6–10 . The proteome is an ideal substrate for the integrated analysis of functional compo- nents owing to proteins’ unmatched diversity, functional roles, and distinct subcellular localization at single-molecule precision. The existing large antibody libraries (close to 100,000 antibodies), once rigorously validated, could enable the detection of more than 70% of the human proteome and corresponding cellular architectures 10 .</p>
<p>For example, multiplexed proteomic imaging techniques (e.g., array tomography and SWITCH) can provide molecular details for individual cells and may allow reconstruction of their surrounding tissue environment 11–13 . Emerging intact tissue clearing approaches preserve proteins and the continuity of neural fibers, which may enable reconstruction of immunolabeled neural architectures 14,15 . Super-resolution imaging of immunolabeled thin tissue sections has been successfully used to characterize minute subcellular structures (e.g., synapses) 16–19 . In addition to conventional super-resolution approaches, Chen et al. 20 have recently demonstrated that protease digestion of a hydrogel-tissue hybrid homogenizes its mechanical characteristics and allows approximately fourfold linear expan- sion of the hybrid. Using this approach, termed expansion micro- scopy (ExM), they demonstrated super-resolution imaging of thin tissue sections with custom-made antibody probes and diffraction- limited microscopes 20 .</p>
<p>Despite the success of these new technologies, integrated recon- struction of the fine subcellular architectures, molecular details, and intercellular connectivity of diverse cell types in large-scale biologi- cal systems remains an unmet goal in biology. For instance, the pro- tease digestion step in ExM causes a loss of proteins, which limits the number of protein structures that can be imaged in the same sample. In addition, intercellular connectivity is largely lost in ExM as it requires tissue sectioning to allow immunolabeling of an unprocessed tissue with limited permeability. By contrast, emerging whole-tissue clearing methods preserve neural connectivity and endogenous proteins, but accurate reconstruction of neural architectures remains challenging owing to the low resolution of diffraction-limited microscopy.</p>
<p>Here we present MAP, a method that enables multiscale proteomic imaging of intact biological systems for combined extraction of the molecular identity, subcellular architectures, and intercellular connectivity of diverse cell types within a single tissue. MAP preserves both the three-dimensional (3D) pro- teome content and organization and the organ-wide cellular connectivity within an intact tissue-hydrogel hybrid, while rendering it reversibly size-adjustable up to four- to fivefold for multiresolution imaging.</p>
<h2 id="results-2"><span class="header-section-number">3.3</span> Results</h2>
<h3 id="organ-scale-tissue-expansion-without-loss-of-proteins"><span class="header-section-number">3.3.1</span> Organ-scale tissue expansion without loss of proteins</h3>
<p>The key to MAP is to prevent intra- and interprotein crosslinking during the hydrogel-tissue hybridization step 21 , and then to dissoci- ate and denature proteins to allow natural expansion of the hybrid (Fig. 1a). We hypothesized that a high concentration of acrylamide monomers might effectively prevent protein crosslinking by quench- ing reactive methylols formed by the protein-formaldehyde reac- tion. If the acrylamide monomer concentration is low, the reactive methylols would react with amide groups within the same protein or adjacent pro- teins to form methylene bridges (Fig. 1a, left column) 22 . Such intra- and interprotein crosslinking would prevent dissociation of protein complexes and limit subsequent tissue expansion 23 . With increased acryla- mide concentration (Fig. 1a, right column), methylols might preferentially react with excess acrylamide monomers, effectively reducing interprotein crosslinking while maximally tethering individual proteins to an expandable hydrogel mesh. Dissociation and denaturation of protein complexes with heat and anionic surfactant then facilitates natural expansion of the hydrogel-tissue hybrid while preserving overall spatial organization of the proteome.</p>
<p>To test our hypothesis, we first measured the effect of acrylamide concentration on tissue expansion. We prepared albumin-containing tissue phantoms and post-fixed them in 4% paraformaldehyde (PFA) with different concentrations (0–20%) of acrylamide. We incubated the tissue phantoms in detergent solution at 95 °C for 1 h to denature and disrupt protein aggregates. As expected, phantoms fixed in higher concentrations of acrylamide showed higher degrees of expansion in water (Fig. 1b). We observed a similar trend in mouse brain tis- sues that were perfused with different concentrations of acrylamide, polymerized, denatured, and expanded (Fig. 1c). Thus, we used high concentrations of acrylamide in order to maximize expansion during MAP in all subsequent experiments.</p>
<p>Using this approach, we achieved a fourfold linear expansion of a whole mouse brain within 7 d without protease treatment (Fig. 1f). Tissue expansion was reversible and tunable using buffers with dif- ferent salt concentrations and osmolarities (Fig. 1d,e). This method is applicable to other organs including heart, lung, spinal cord, liver, intestine, and kidney, and also cerebral organoids (Fig. 1g,h and Supplementary Fig. 1).</p>
<h3 id="preservation-of-multiscale-architectures"><span class="header-section-number">3.3.2</span> Preservation of multiscale architectures</h3>
<p>We next asked whether MAP retains multiscale structural informa- tion and enables super-resolution imaging with diffraction-limited microscopes. To estimate the amount of distortion incurred from expansion, we imaged gel-embedded cultured cells before and after MAP processing (Fig. 2a). At the subcellular scale, MAP expansion improved visualization of microtubules and allowed imaging of single tubular structures (Fig. 2b,c). The estimated distortion error (root- mean-square error, RMSE) was less than 3% of measured length at both the subcellular scale (Fig. 2d) and the multicellular scale (Fig. 2g). The degree of gel expansion was not a function of cell density, as indi- cated by the consistent local expansion within differently populated cell clusters (Fig. 2e,f).</p>
<p>To test whether MAP preserves multiscale tissue architectures, we imaged a 100-μm-thick mouse brain block labeled with lectin before and after MAP (Fig. 2h). The distortion analysis showed less than 4% error (Fig. 2i), which was at most a twofold increase, com- pared to the inevitable distortion from sample mounting for imaging (Fig. 2i, remounting). Both the intra-vascular space with low protein concentration and the extra-vascular space with high protein con- centration showed similar degrees of expansion (Fig. 2h, insets), sug- gesting that the effect of protein concentration and cell density on gel expansion is minimal. Diffraction-limited microscopy revealed fine 3D details of immunolabeled cells such as cytoskeletal filament structures (Fig. 2j,k,m) and better-resolved axonal fibers (Fig. 2l) after the MAP process. These results together indicate that MAP preserved multiscale structural information of biological samples and enabled super-resolution imaging with diffraction-limited microscopes, achieving ~60-nm lateral resolution.</p>
<h3 id="labeling-proteome-with-conventional-antibodies"><span class="header-section-number">3.3.3</span> Labeling proteome with conventional antibodies</h3>
<p>We next asked if MAP preserves the 3D proteome composition and organization, and enables super-resolution imaging of fine subcellular architectures using commercially available antibodies. Antibody tar- gets were selected from a wide range of antigens including membrane proteins, cytoplasmic proteins, nuclear proteins, neurofilament pro- teins, and synaptic proteins to survey the overall proteomic landscape (Fig. 3 and Supplementary Table 1). To validate the antibodies, we tested whether antibody staining of MAP-processed tissues shows comparable structures to PFA-fixed tissues, and antibodies from dif- ferent host species targeting the same protein, if available, provide concordant images (Supplementary Figs. 2 and 3). We found that 100 of 122 antibodies were compatible with MAP-processed samples, and 43 of 51 target molecules were successfully visualized (Fig. 3, Supplementary Table 1, and Supplementary Figs. 2 and 3). This high success rate indicates that loss of epitopes by protein denatura- tion was minimal. Moreover, some antibodies (e.g., calbindin and calretinin) showed negative staining when different antibodies tar- geting the same endogenous protein showed strong signal (Fig. 3b and Supplementary Table 1). This result demonstrates that MAP preserves the tissue proteome with minimal loss of specific epitopes and that the library can be effectively probed using off-the-shelf anti- bodies without any modification.</p>
<h3 id="super-resolution-imaging-of-3d-subcellular-architectures"><span class="header-section-number">3.3.4</span> Super-resolution imaging of 3D subcellular architectures</h3>
<p>Preservation of both nanoscopic structures and the proteome in MAP enables super-resolution imaging of a broad range of fine subcellular architectures. Antibodies targeting filament proteins successfully visualized details of the cytoskeletal networks of various cell types (Fig. 3c–e and Supplementary Videos 1–4). No apparent filament fragmentations were observed in the magnified tissues. Labeling of glial fibrillary acidic protein (GFAP) enabled visualization the fine foot-process structures surrounding blood vessels (Fig. 3e and Supplementary Video 3) 24,25 . Neurofilament medium subunit (NF-M) enabled visualization of both processes and synapses 26 , a subset of which co-localized with the postsynaptic marker, GABA B R1 (the C-terminal domain of the GABA B receptor subunit-1) (Fig. 3f and Supplementary Video 4). Labeling of calcium-binding proteins (parvalbumin, calbindin, and calbindin) robustly enabled visuali- zation of morphological details of positive cells (Fig. 3b). A small subset of calbindin-positive fibers showed expression of calbindin in their dendritic spines (Fig. 3g). Many synaptic protein markers were compatible with MAP (Fig. 3f,h,j,l,m and Supplementary Table 1). Staining of synaptic proteins clearly visualized distinct and well-separated elliptical disk-shaped clusters of pre- and post-synaptic proteins (Fig. 3h,l,m and Supplementary Video 5), enabling their quantitative analysis (Fig. 3k).</p>
<h3 id="highly-multiplexed-immunolabeling"><span class="header-section-number">3.3.5</span> Highly multiplexed immunolabeling</h3>
<p>Another potential advantage of the MAP technology is that it enables highly multiplexed labeling and imaging of the magnified 3D pro- teome within a single tissue. We successfully performed seven rounds of immunostaining of a MAP-processed 100-μm-thick mouse brain tissue with no obvious signs of tissue damage (Fig. 4). Outstanding mechanical stability of the MAP-processed sample enabled repeated manual handling of the tissue with tools that are widely used in biol- ogy laboratories. The images after destaining of the imaged anti-GFP antibody (first round) and after solely adding secondary antibodies targeting the eluted anti-GFP antibody (second round) showed lit- tle signal, suggesting that the destaining process effectively eliminates antibodies. For the following five rounds of labeling, we continued to use anti-GFP as a landmark in addition to two other antibodies for each round. The consistent GFP signals show that the same antigen can be repeatedly labeled without loss of antigenicity. Successful visualization of ten other targets demonstrated that MAP facilitated exploration of diverse proteins, structures, and cell types within a single tissue.</p>
<h3 id="maping-neural-projections"><span class="header-section-number">3.3.6</span> MAPing neural projections</h3>
<p>We next explored the potential utility of MAP for mapping inter-areal neural connectivity. If MAP could preserve continuity of neural fibers within magnified, intact brain tissue along with its 3D proteome, highly specific antibodies would enable visualization of a subset of neural projections connecting different brain regions. The sparse labeling and enhanced spatial reso- lution offered by MAP may allow more accurate reconstruction of the labeled pro- jections. In addition, highly multiplexed proteomic imaging may enable reconstruc- tion of many different cell types with integrated molecular and fine morphological details in a single tissue.</p>
<p>To explore this possibility, we first asked if the continuity of neural processes is preserved within a magnified sample. We expanded a 0.5- mm-thick Thy1-eGFP mouse brain coronal block and then imaged GFP-labeled neurons and their projections 27 . MAP preserved the continuity of the GFP-expressing projections spanning a large tis- sue volume (Fig. 5a–f) as well as their fine morphological details (e.g., dendritic spines) (Fig. 5g). Immunolabeling of various cyto- plasmic proteins including neurofilament proteins (e.g., NF-H and SMI-32), calcium-binding proteins, and metabolic enzymes (e.g., tyrosine hydroxylase) confirmed that MAP preserved the continuity of neural fibers of various cell types (Fig. 5h–n and Supplementary Videos 6–8). The preserved continuity enabled detection of single fibers selected in the immunostained tissue vol- umes (Fig. 5m–q and Supplementary Videos 7 and 8).</p>
<p>To test if MAP enables more accurate tracing of densely packed fibers, we imaged neurofilament-stained samples before and after MAP processing using a high numerical aperture (NA; 0.95) water-immersion objective. Two individuals not involved in image acquisition traced fibers within the sample volume (Fig. 5r,s). When we compared the concordance between the two tracing results, the discordant rate was significantly lower after MAP (Fig. 5t,u; SMI-312, n = 4, P = 0.020; NF-H, n = 4, P = 0.010), even though MAP enabled the tracers to detect more fibers (total numbers of traced fibers were 160 before MAP and 214 after MAP). This result demonstrates that MAP indeed enabled more accurate reconstruction of immunolabeled neural fibers.</p>
<h3 id="immunolabeling-and-imaging-of-mm-thick-tissues"><span class="header-section-number">3.3.7</span> Immunolabeling and imaging of mm-thick tissues</h3>
<p>Reconstruction of individual neurons requires labeling and imaging of thick brain tissues because nerve fibers can extend across a large volume. To test whether MAP is applicable to large-scale brain tissues, we expanded a 1-mm-thick mouse brain block (5-mm thick after expansion, Fig. 6b) and passively stained it with anti-GFP antibody (Fig. 6a). We then imaged the sample using both a high NA (0.95), short working distance (WD) (2 mm) water-immersion objective and a low NA (0.6), long WD (8 mm) CLARITY objective. Both objectives showed fine dendritic spines of GFP-expressing neurons throughout the entire volume (Fig. 6e). Imaging of the sample did not require depth-dependent modulation of the laser power, indicating that staining was uniform and that signal attenuation by light scattering was negligible (Fig. 6c,d). MAP is also compatible with stochastic electrotransport 28 , a method that enables rapid tissue labeling. Using stochastic electrotransport, we were able to label an 8-mm-thick expanded tissue uniformly within only 2 d (Fig. 6f–j). The sample was highly transparent. We were able to image the 8-mm-thick sample successfully with the same laser power up to the working distance of the objective (Fig. 6h).</p>
<h2 id="discussion-2"><span class="header-section-number">3.4</span> Discussion</h2>
<p>As a step toward organ-scale reconstruction of diverse cell types and their surrounding environment and to advance the study of com- plex system-level interactions, we developed a simple and scalable method that enables preservation, reversible expansion, and imaging of the 3D proteome organization within an intact tissue. We discov- ered that the protein content of a whole organ can be preserved and magnified by preventing intra- and interprotein crosslinking, then denaturing and dissociating the protein complexes to allow natural expansion of hydrogel-tissue hybrids. The expanded hybrids secure both fine subcellular architectures and organ-scale cellular connec- tivity. These multiscale properties can be directly imaged by using off-the-shelf antibodies to label the structures’ constituent proteins. The high success rate that we achieved with commercial antibodies might be because synthetic peptides or denatured protein fragments are commonly used for conventional antibody production 29 . An exist- ing large antibody library, once validated, can be used without any modification. This technique is easy to implement, and it does not require any special equipment or chemicals.</p>
<p>MAP enables repeated interrogation of the same expanded speci- men. This is possible because the preserved epitopes, which have already reached complete denaturation by harsh treatment (95 °C, 200 mM sodium dodecyl sulfate (SDS)) for expansion, do not undergo any substantial modification during the milder antibody elution step (70 °C, 200 mM SDS). We performed seven rounds of staining of the same tissue without any signs of tissue damage. However, the practical limit of this approach must be carefully examined.</p>
<p>MAP has the potential to enable scalable super-resolution imaging of large-scale samples. Unlike other methods, thick tissues can be labeled on a practical time scale because antibody-labeling occurs after complete lipid removal and tissue permeabilization. The great transparency of the MAP-processed samples allows high-resolution imaging of the physically expanded tissue with minimal loss of reso- lution. Currently, 1.0 NA, 2.5-mm WD water-immersion objectives are best suited for MAP. Although the 1.0 NA, 8-mm WD CLARITY objective has the longest WD, it does not provide high-quality images because it is optimized for immersion media with a high refrac- tive index (RI). Further development of high-NA, long-WD water objectives or the development of high-RI immersion media that are compatible with MAP would extend the utility of MAP. Another chal- lenge in the MAP approach is the dilution of fluorescent signals that accompanies physical volumetric expansion. Fourfold linear expan- sion decreases signal density by 64-fold. Therefore, much higher laser power is required, which in turn causes photobleaching. Future studies will need to explore the compatibility of signal amplification techniques with MAP.</p>
<p>The reversible modulation of physical sample size that is attained with MAP enables multiscale proteomic imaging of a single tissue to capture both system-scale properties and fine local details in a practical way. For example, following sample shrinkage, global projection pat- terns of labeled neurons may be imaged using high-speed microscopy techniques and long WD objectives (e.g., 25×, 1.0 NA, 8-mm WD objective, currently available) 30 . After imaging, the same tissue can be expanded for super-resolution imaging of regions of interest. The expanded tissue may need to be sliced before imaging to meet the objective’s WD limit. A microscope with a built-in vibrating-blade microtome could also be used for whole-mount imaging to avoid loss of connectivity information 31–33 . This approach may allow us to drastically reduce the imaging time and costs associated with data storage and handling of expanded samples.</p>
<p>The advantages of MAP (Supplementary Table 2) may allow combined extraction of rich molecular details, minute subcellular architectures, and cellular connectivity from diverse cell types within a single tissue. Together with its simplicity and broad applicability, MAP may complement existing methods and enable new approaches in the study of complex biological systems.</p>
<h2 id="methods-1"><span class="header-section-number">3.5</span> Methods</h2>
<p>Methods and any associated references are available in the online version of the paper.</p>
<p>Note: Any Supplementary Information and Source Data files are available in the online version of the paper.</p>
<h2 id="acknowledgements-2"><span class="header-section-number">3.6</span> Acknowledgements</h2>
<p>The authors thank the entire Chung laboratory for support and helpful discussions. K.C. was supported by Burroughs Wellcome Fund Career Awards at the Scientific Interface, the Searle Scholars Program, Packard award in Science and Engineering, JPB Foundation (PIIF and PNDRF) and NIH (1-U01-NS090473-01). Resources that may help enable general users to establish the methodology are freely available online (http://www.chunglabresources.org). K.C. is a co-founder of LifeCanvas Technologies, a startup that aims to help the research community adopt technologies developed by the Chung Laboratory.</p>
<h2 id="author-contributions-2"><span class="header-section-number">3.7</span> Author Contributions</h2>
<p>T.K., J.S., J.-Y.P., and K.C. designed the experiments and wrote the paper with input from other authors. T.K. stained and imaged mouse samples. J.S. performed the gel and cell experiments. T.K. and J.S. analyzed the data. J.-Y.P. prepared mouse tissues. J.-Y.P. and V.M. processed mouse MAP samples. A.A. performed the cell and organoid experiments. E.M., Y.-G.P., and T.K. performed the antibody validation test. J.H.C. performed stochastic electrotransport staining. Y.-G.P. and T.K. obtained synaptic images. J.-Y.P., V.M., T.K., and J.S. performed tracing. J.C. performed the gel experiment. K.C. supervised all aspects of the work.</p>
<h2 id="competing-financial-interests"><span class="header-section-number">3.8</span> Competing Financial Interests</h2>
<p>The authors declare competing financial interests: details are available in the online version of the paper.</p>
<h2 id="references-2"><span class="header-section-number">3.9</span> References</h2>
<p>1.White, J.G., Southgate, E., Thomson, J.N. &amp; Brenner, S. The structure of the nervous system of the nematode Caenorhabditis elegans. Philos. Trans. R. Soc. Lond., B314, 1–340 (1986).</p>
<p>2.DeFelipe, J. From the connectome to the synaptome: an epic love story. Science330, 1198–1201 (2010).</p>
<p>3.Helmstaedter, M. Cellular-resolution connectomics: challenges of dense neural circuit reconstruction. Nat. Methods10, 501–507 (2013).</p>
<p>4.Craddock, R.C. et al. Imaging human connectomes at the macroscale. Nat. Methods10, 524–539 (2013).</p>
<p>5.Oh, S.W. et al. A mesoscale connectome of the mouse brain. Nature508, 207–214 (2014).</p>
<p>6.Phizicky, E., Bastiaens, P.I., Zhu, H., Snyder, M. &amp; Fields, S. Protein analysis on a proteomic scale. Nature422, 208–215 (2003).</p>
<p>7.Hanash, S. HUPO initiatives relevant to clinical proteomics. Mol. Cell. Proteomics3, 298–301 (2004).</p>
<p>8.Agaton, C., Uhlén, M. &amp; Hober, S. Genome-based proteomics. Electrophoresis25, 1280–1288 (2004).</p>
<p>9.Berglund, L. et al. A genecentric Human Protein Atlas for expression profiles based on antibodies. Mol. Cell. Proteomics7, 2019–2027 (2008).</p>
<p>10.Uhlen, M. et al. Towards a knowledge-based human protein atlas. Nat. Biotechnol.28, 1248–1250 (2010).</p>
<p>11.Micheva, K.D. &amp; Smith, S.J. Array tomography: a new tool for imaging the molecular architecture and ultrastructure of neural circuits. Neuron55, 25–36 (2007).</p>
<p>12.Kay, K.R. et al. Studying synapses in human brain with array tomography and electron microscopy. Nat. Protoc.8, 1366–1380 (2013).</p>
<p>13.Murray, E. et al. Simple, scalable proteomic imaging for high-dimensional profiling of intact systems. Cell163, 1500–1514 (2015).</p>
<p>14.Dodt, H.-U. et al. Ultramicroscopy: three-dimensional visualization of neuronal networks in the whole mouse brain. Nat. Methods4, 331–336 (2007).</p>
<p>15.Richardson, D.S. &amp; Lichtman, J.W. Clarifying tissue clearing. Cell162, 246–257 (2015).</p>
<p>16.Rust, M.J., Bates, M. &amp; Zhuang, X. Sub-diffraction-limit imaging by stochastic optical reconstruction microscopy (STORM). Nat. Methods3, 793–795 (2006).</p>
<p>17.Betzig, E. et al. Imaging intracellular fluorescent proteins at nanometer resolution. Science313, 1642–1645 (2006).</p>
<p>18.Hess, S.T., Girirajan, T.P. &amp; Mason, M.D. Ultra-high resolution imaging by fluorescence photoactivation localization microscopy. Biophys. J.91, 4258–4272 (2006).</p>
<p>19.Maglione, M. &amp; Sigrist, S.J. Seeing the forest tree by tree: super-resolution light microscopy meets the neurosciences. Nat. Neurosci.16, 790–797 (2013).</p>
<p>20.Chen, F., Tillberg, P.W. &amp; Boyden, E.S. Optical imaging. Expansion microscopy. Science347, 543–548 (2015).</p>
<p>21.Chung, K. &amp; Deisseroth, K. CLARITY for mapping the nervous system. Nat. Methods10, 508–513 (2013).</p>
<p>22.Puchtler, H. &amp; Meloan, S.N. On the chemistry of formaldehyde fixation and its effects on immunohistochemical reactions. Histochemistry82, 201–204 (1985).</p>
<p>23.Sung, H.-W., Hsu, H.-L., Shih, C.-C. &amp; Lin, D.-S. Cross-linking characteristics of biological tissues fixed with monofunctional or multifunctional epoxy compounds. Biomaterials17, 1405–1410 (1996).</p>
<p>24.Kosaka, T. &amp; Hama, K. Three-dimensional structure of astrocytes in the rat dentate gyrus. J. Comp. Neurol.249, 242–260 (1986).</p>
<p>25.Khakh, B.S. &amp; Sofroniew, M.V. Diversity of astrocyte functions and phenotypes in neural circuits. Nat. Neurosci.18, 942–952 (2015).</p>
<p>26.Yuan, A. et al. Neurofilament subunits are integral components of synapses and modulate neurotransmission and behavior in vivo. Mol. Psychiatry20, 986–994 (2015).</p>
<p>27.Feng, G. et al. Imaging neuronal subsets in transgenic mice expressing multiple spectral variants of GFP. Neuron28, 41–51 (2000).</p>
<p>28.Kim, S.Y. et al. Stochastic electrotransport selectively enhances the transport of highly electromobile molecules. Proc. Natl. Acad. Sci. USA112, E6274–E6283 (2015).</p>
<p>29.Uhlen, M. &amp; Ponten, F. Antibody-based proteomics for human tissue profiling. Mol. Cell. Proteomics4, 384–393 (2005).</p>
<p>30.Tomer, R., Ye, L., Hsueh, B. &amp; Deisseroth, K. Advanced CLARITY for rapid and high-resolution imaging of intact tissues. Nat. Protoc.9, 1682–1697 (2014).</p>
<p>31.Ragan, T. et al. Serial two-photon tomography for automated ex vivo mouse brain imaging. Nat. Methods9, 255–258 (2012).</p>
<p>32.Economo, M.N. et al. A platform for brain-wide imaging and reconstruction of individual neurons. eLife5, e10566 (2016).</p>
<p>33.Albanese, A. &amp; Chung, K. Whole-brain imaging reaches new heights (and lengths). eLife5, e13367 (2016).</p>
<h2 id="methods-2"><span class="header-section-number">3.10</span> Methods</h2>
<h3 id="bsa-hydrogel-denaturation-and-expansion"><span class="header-section-number">3.10.1</span> BSA hydrogel denaturation and expansion</h3>
<p>Stock solutions of 40% BSA, 40% acrylamide (AA), 32% paraformaldehyde (PFA), and 1% VA-044 were made and kept on ice throughout the experiment. A 10-mL solution of 4% BSA, 4% AA, 4% PFA, 0.1% bis-acrylamide (BA), and 0.1% VA-044 was made in phosphate-buffered saline (PBS). The solution was polymerized under vacuum at 37 °C for 2 h, and the resulting albumin-containing tissue phantom was sectioned. Individual tissue phantom sections were washed in excess PBS with shaking for 12 h. After washing, four phantoms were placed in 10 mL of each of the following PBS solutions for 4 h at 37 °C: 2% glutaraldehyde (GA), 4% PFA, 4% AA with 4% PFA, and 20% AA with 4% PFA. Tissue phantoms were washed similarly, massed, photographed, and then incubated in a 200-mM SDS solution with 50 mM sodium sulfite for 1 h at 95 °C. Phantoms were washed again and incubated in deionized (DI) water for 12 h. After expansion, phantoms were massed and photographed.</p>
<h3 id="general-map-protocol"><span class="header-section-number">3.10.2</span> General MAP protocol</h3>
<ol type="i">
<li><p>Perfusion and hydrogel embedding. Thy1-eGFP-M mice (6–8 weeks old, male and female) were housed in a reverse 12-h light/dark cycle with unrestricted access to food and water. All experimental protocols were approved by the MIT Institutional Animal Care and Use Committee and the Division of Comparative Medicine and were in accordance with guidelines from the US National Institutes of Health. After anesthesia, the mice were first washed transcardially with a mixture of 2–5% AA, 0–0.05% BA, 0–0.8% sodium acrylate (SA), and PBS, followed by perfusion with a mixture of 4% PFA, 30% AA, 0.05–0.1% BA, 10% SA, 0.1% VA-044 or V-50, and PBS. The perfusion solution could be slightly turbid. Upper transparent solution was used after centrifugation during 3 min with 1,000g. All solution was protected from light and kept on ice before perfusion. Control samples were perfused first with PBS and then with 4% PFA and PBS. The brain and other organs (heart, lung, liver, intestine, kidney, and spinal cord) were harvested and incubated in 20–40 mL of the same fixative solution at 4 °C for 2–3 d and then for 1–3 d at room temperature (RT) with gentle shaking to ensure uni- form chemical diffusion and reaction throughout the sample. Following the diffusion and fixation steps, hydrogel-tissue hybridization was performed in situ by incubating the tissues using Easy-Gel (LifeCanvas Technologies) with nitrogen gas at 45 °C for 2 h.</p></li>
<li><p>Tissue denaturation. Hydrogel-embedded tissues were incubated overnight in a solution of 200 mM SDS, 200 mM NaCl, and 50 mM Tris in DI water (pH titrated to 9.0) at 37 °C with gentle shaking. The samples were then incubated at 70 °C for 0–50 h and 95 °C for 1–24 h depending on their size using EasyClear (LifeCanvas Technologies). Whole organs were incubated for 24–48 h at 70 °C followed by 12–24 h at 95 °C. 1-mm-thick brain slices were incubated for 5 h at 70 °C and then at 95 °C for 1 h.</p></li>
<li><p>Expansion. Denatured tissues were incubated in 40–100 mL DI water at RT for 12–48 h with gentle shaking. During DI water incubation, the solution was changed every 3–5 h.</p></li>
</ol>
<h3 id="expansion-according-to-various-aa-and-sa-concentrations"><span class="header-section-number">3.10.3</span> Expansion according to various AA and SA concentrations</h3>
<p>After anesthesia, mice were washed transcardially with 4% AA, 0.05% BA, 0.8% SA, 0.1% VA-044, and PBS and then with 4% PFA, 0.05% BA, 0.1% VA-044, one of four AA and SA combinations (5% AA + 0.8% SA, 10% AA + 1.7% SA, 20% AA + 3.3% SA, and 30% AA + 5% SA) in PBS. Tissues were incubated in 20 mL of the same fixative solution at 4 °C for 2 d and 5 h at RT with gentle shaking. After hydrogel-tissue hybridization the samples were incubated in denaturation solution at 70 °C for 5 h and 95 °C for 1 h with gentle shaking. Denatured tissues were incubated in 40 mL DI water at RT for 12 h with gentle shaking. During DI water incubation, the solution was changed every 3–5 h. Fiji (National Institutes of Health) was used to measure the size of the expanded samples 34 .</p>
<h3 id="shrinkage-and-ri-matching"><span class="header-section-number">3.10.4</span> Shrinkage and RI matching</h3>
<p>A customized RI matching solution was made by dissolving 50 g diatrizoic acid, 40 g N-methyl-d-glucamine, and 55 g iodixanol per 100 mL PBS 28 . The RI was targeted to 1.47. This solution was used for both shrinkage and imaging. Depending on sample size, samples were incubated in 1–10 mL solution at RT for 2–24 h with gentle shaking. The solution was changed every 1–12 h. For imaging tissues before MAP processing, samples were incubated in 1–10 mL of this solution without PBS at RT with gentle shaking for 2–5 h before imaging.</p>
<h3 id="map-processing-of-cerebral-organoid"><span class="header-section-number">3.10.5</span> MAP processing of cerebral organoid</h3>
<p>Cerebral organoids were made from stem cells following a previously described protocol 35 . Organoids were ini- tially fixed in 4% PFA for 15 min, incubated in a mixture of 4% PFA, 30% AA, 0.1% BA, 10% SA, 0.1% V-50, and PBS for 24 h at 4 °C, followed by 24 h, at RT. Hydrogel embedding, tissue denaturation, and expansion were processed similarly to “General MAP protocol.”</p>
<h3 id="cultured-cell-experiment"><span class="header-section-number">3.10.6</span> Cultured cell experiment</h3>
<p>For tubulin imaging in HeLa cells, 8-mm round glass coverslips were coated in 0.1% gelatin in ultrapure water (Millipore). Coverslips were placed in a 48-well plate and seeded with 50,000 HeLa cells overnight. To obtain comparable images before and after MAP processing, cells were washed, fixed with 3% PFA + 0.1% GA in PBS for 10 min, and switched to a solution of 4% PFA, 30% AA in PBS for 8 h at 37 °C. Cells were then placed in 0.1% sodium borohydride for 7 min at RT then incubated in 100 mM glycine for 10 min at RT. Cells were washed and stained with anti-tubulin (Abcam, ab6160), Alexa Fluor 594-conjugated secondary (Abcam, ab150152) antibodies and TOTO-1 (Thermo Fisher Scientific). Cells were mounted in 2,2′-thiodiethanol (Sigma) and imaged with a 63×, 1.3 NA glycerol-immersion objective with the Leica microscope system. Cells were washed extensively and embedded into a MAP hybrid polymer by addition of 20 μL of Cell-MAP solution (20% AA, 7% SA, 0.1% BA, 0.5% TEMED, 0.5% ammonium persulfate in PBS). Ammonium persulfate was added last from a freshly prepared 5% stock solution. Cell-MAP solution was quickly added to the coverslip and left to polymerize for 4–5 min. Gels were peeled off the coverslip using forceps, washed extensively and denatured for 30 min in denaturation buffer at 95 °C. Cell-MAP gels were washed extensively, restained with anti-tubulin antibody and TOTO-1 and reimaged.</p>
<h3 id="immunostaining-of-brain-tissue"><span class="header-section-number">3.10.7</span> Immunostaining of brain tissue</h3>
<p>For typical staining, MAP-processed 100- to 500-μm-thick mouse brain coronal slices were incubated with primary anti- bodies (typical dilution, 1:100) in PBS with 1% (wt/vol) Triton X-100 (PBST) at 37 °C for 8–16 h, followed by washing at 37 °C for 1–2 h in PBST three times. The tissue was then incubated with secondary antibodies (typical dilution, 1:100) in PBST at 37 °C for 6–16 h, followed by washing at 37 °C for 1–2 h in PBST three times. For antibody validation of a given antibody, 100-μm-thick PFA-fixed control and MAP-processed samples were stained with the same titer of primary and, if necessary, secondary antibodies overnight in PBST. See Supplementary Table 1 for the list of antibodies used. To destain for multi- plexed labeling, samples were incubated in a denaturation solution 6–16 h at 70 °C, and washed with PBST at 37 °C for 1–2 h three times.</p>
<h3 id="mounting-and-imaging"><span class="header-section-number">3.10.8</span> Mounting and imaging</h3>
<p>Samples were mounted on a slide glass. Blu- Tack adhesive was applied on the Petri dish or the slide glass, and samples were covered with a glass-bottom Willco dish. The space between the bot- tom material and the Willco dish around the sample was filled with either shrinkage solution or DI water according to the sample immersion medium. Large expanded samples were additionally placed on a 120-mm-diameter Petri dish, and the dish was filled with DI water. Expanded or shrunk sam- ples were stabilized for at least 1 h before imaging. Samples were imaged with either the Olympus FV1200MPE microscope system or the Leica TCS SP8 microscope system. A 10×, 0.6 NA CLARITY-optimized objec- tive (XLPLN10XSVMP; 8.0-mm WD) was used with the Olympus system to obtain wide-field images of shrunk samples and z-stack images of large samples. The images of MAP-processed samples were obtained with a 10×, 0.3 NA water-immersion objective, a 20×, 0.95 NA water-immersion objec- tive, and a 40×, 1.25 NA oil-immersion objective with the Olympus system, or a 10×, 0.3 NA water-immersion objective, a 25×, 0.95 NA water-immersion objective, and 63×, 1.30 NA glycerol-immersion objective with the Leica system. Single-photon confocal laser scanning imaging was performed with 405-, 488-, 559-, and 635-nm lasers (Olympus) or a white-light laser (Leica). Mai Tai DeepSee (Spectra-Physics) was used for multi-photon excitation with 780-nm wavelength. The images were visualized and analyzed with Fiji or Imaris (Bitplane).</p>
<h3 id="large-tissue-staining"><span class="header-section-number">3.10.9</span> Large tissue staining</h3>
<p>1- and 2-mm-thick mouse brain coronal slices were prepared by “General MAP protocol,” and expanded. A 1-mm-thick slice was chopped to about 3 mm × 3 mm × 1 mm (dimensions before MAP). The sample was stained passively with Alexa Fluor 594-conjugated rabbit anti-GFP antibody (Life Technologies, A21312) for 3 d and Alexa Fluor 594-conjugated donkey anti-rabbit IgG antibody (Abcam, ab150072) for 3 d. 20 μL of antibody was used in 500 μL PBST, and was washed with 40 mL PBST for 1.5 d (three times in total) for each antibody. We used stochastic electrotransport 28 to stain a 2-mm-thick slice with Alexa Fluor 647-conjugated rabbit anti-GFP antibody (Life Technologies, A31852) and Alexa Fluor 647-conjugated don- key anti-rabbit IgG antibody (Abcam, ab181347). For each antibody, we first electrotransported stochastically a solution containing 20 μL of antibody in 4 mL of 0.6 M N-cyclohexyl-3-aminopropanesulfonic acid (CAPS), 0.2 M Tris, 100 mM NaCl, 20 mM SDS, 1% BSA with an electrophoresis buffer contain- ing 0.3 M CAPS, 0.2 M Tris, 20 mM SDS, 30% sorbitol for 21 h. It was then electrotransported stochastically in a solution containing 4 mL of 0.3 M CAPS, 0.2 M Tris, 20 mM SDS, 30% sorbitol, 1% BSA, and 1% Triton X-100 with an electrophoresis buffer containing 0.04 M Tris, 0.01 M phosphate, 30% sorbitol for 8 h. These two steps complete the stochastic electrotransport labeling. The stained sample was expanded in a solution containing 0.01% (wt/vol) heparin sodium porcine mucosa (Sigma, SRE0027) and 1% Triton X-100 in DI water, and then imaged.</p>
<h3 id="cell-distortion-analysis"><span class="header-section-number">3.10.10</span> Cell distortion analysis</h3>
<p>Before and after MAP processing tubulin images were first registered using a scaled-rotation transformation in Fiji. Non-rigid invertible B-spline registration was performed with an 8 × 8 control point grid in bUnwarpJ. Vectors of different length were subjected to the resulting nonlinear transformation, and the input-output difference norm was sorted based on the input vector length and then averaged by root-mean-square. For morphological image processing of nuclei, TOTO-1 images before and after MAP-processing were first registered using a scaled-rotation transformation in Fiji. The registered images were segmented by thresholding and converted to circular particles with equivalent average radii calculated from the “Analyze Particles” function in Fiji. Matched pairs of cells from the before and after images were randomly chosen, and the expansion ratio was calculated from the ratio of the connecting line segment lengths. The fraction in nuclei was obtained by summing the intensity profile along the connecting line segment and averaging the sums from the before and after binary masks.</p>
<h3 id="tissue-distortion-analysis"><span class="header-section-number">3.10.11</span> Tissue distortion analysis</h3>
<p>After anesthesia, mice were first washed transcar- dially with 2% AA in PBS followed by perfusion with 4% PFA and 30% AA in PBS. Thy1-eGFP-M mouse brains were harvested and incubated in 20 mL of the same fixative solution at 4 °C overnight and at 37 °C for 3 h. Brains were sectioned to 100-μm-thick coronal slices with a vibrating microtome. Slices were stained and imaged to obtain “before MAP” images, and then incubated in a solution containing 4% PFA, 30% AA, 0.1% BA, 10% SA, and 0.1% V-50 in PBS at RT for 8 h. Hydrogel-tissue hybridization was performed in situ by incubating the tissues with nitrogen gas at 45 °C for 2 h. Hydrogel-embedded tissues were incubated in denaturation solution at 37 °C for 1 h and 95 °C for 0.5–1 h. Samples were then stained with the same markers and imaged to obtain “after MAP” images.</p>
<p>To quantify distortion errors, regions of ~3 mm × 2.5 mm in size that included cortex and hippocampus were stained with DyLight 594-conjugated lectin (Vector Laboratories, DL-1177) before (8 μL in 200 μL PBST for up to 8 samples) and after (2 μL in 200 μL PBST for each sample) MAP processing. Samples were incubated in RI matching solution after staining and mounted and imaged before MAP processing. Five samples were repeated for incubation, mounting and imaging to measure mounting errors. After MAP processing, six samples were stained, expanded in DI water, and imaged. Keypoints of the vasculature in volumetric images were detected and matched between two image sets with a MATLAB code implementing the 3D Harris Corner Detector and 3D SIFT algorithm as described previously 13 . Using custom-built graphical user interface software developed with Delphi XE4 (Embarcadero Technologies), redundant keypoints closely located to each other and keypoints at tissue margins were removed. Tissue sizes were estimated by the area defined by a convex hull encompassing all key-points, and the expansion ratio was calculated as the ratio between two squared roots of the areas. The correspondence information was used to generate a regularly spaced deformation mesh using a 3D thin plate spline code written by Yang (http://www.mathworks.com/matlabcentral/fileexchange/37576-3d-thin-plate-spline-warping-function). Lengths between each pair of grid points were calculated in both pre-MAP and post-MAP images, considering the expansion ratio. The difference between the two lengths was measured as a distortion error. After averaging the squared errors for each measurement length, the square root of the averages was collected from the samples to obtain statistical values of error.</p>
<h3 id="neurofilament-tracing"><span class="header-section-number">3.10.12</span> Neurofilament tracing</h3>
<p>Tracing of individual neurons was performed using either Fiji or Imaris. For manual tracing of a long tyrosine hydroxylase fiber, a representative fiber was chosen during confocal imaging acquisition and traced by moving the motorized stage and adjusting the z-level. During the tracing, ambiguous crossovers were resolved by obtaining high-magnification subvolume images. After tile-scanning, the target fiber was re-identified from the image volume using Fiji and marked to be displayed in a two-dimensional plane. For the semi-automatic tracing of SMI-312 fibers in a dense region, the entire 800 μm × 800 μm × 150 μm (expanded) data set was loaded along with a filament tool into a ‘Surpass’ instance using Imaris. An autopath calculation was performed using a single starting point as indicated in Supplementary Video 8. The fiber endpoint was designated by selecting the portion of the fiber exiting the imaged volume. The fiber representation was changed to a cone representation to visualize the filament diameter as well as the tracing path. The tracing fidelity was confirmed by inspection. To trace multiple fibers in a dense region, the full data set was cropped to the region indicated in Figure 5o using the ‘3D crop’ tool, and similar autopath calculations were performed for each fiber. For traceability comparison between before and after MAP, we imaged the same tissues before and after MAP processing at the same resolution near the optical sampling limit with the Leica system and a 25×, 0.95 NA water-immersion objective. Two individuals not involved in imaging acquisition performed manual tracing using autodepth assistance in Imaris, and discordance ratios between manual tracing results before and after MAP processing were calculated.</p>
<h3 id="synaptic-and-fiber-intensity-profiles"><span class="header-section-number">3.10.13</span> Synaptic and fiber intensity profiles</h3>
<p>The regions of interest were imported into Fiji. Lines were drawn perpendicular to the synaptic junction or near the fibers of interest and intensity profiles were obtained with adjusting the line width. For synaptic intensity profiles, two Gaussian distributions were fit to the distinct peaks by simultaneous minimization of the sum of squared residuals. Synaptic densities for bassoon and VGluT2 were calculated from three non-overlapping xy-images (235 μm × 235 μm) of expanded samples. The images were segmented by thresholding, and individual synaptic structures for each channel were counted using the “Analyze Particles” function in Fiji. The synaptic densities were calculated based on the frame area, and the s.d. was calculated from the three replicates.</p>
<p>34.Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods9, 676–682 (2012).</p>
<p>35.Lancaster, M.A. &amp; Knoblich, J.A. Generation of cerebral organoids from human pluripotent stem cells. Nat. Protoc.9, 2329–2340 (2014)</p>
<h1 id="research-containing-a-figure"><span class="header-section-number">4</span> Research containing a figure</h1>
<h2 id="introduction-3"><span class="header-section-number">4.1</span> Introduction</h2>
<p>This is the introduction. Sed vulputate tortor at nisl blandit interdum. Cras sagittis massa ex, quis eleifend purus condimentum congue. Maecenas tristique, justo vitae efficitur mollis, mi nulla varius elit, in consequat ligula nulla ut augue. Phasellus diam sapien, placerat sit amet tempor non, lobortis tempus ante.</p>
<h2 id="method"><span class="header-section-number">4.2</span> Method</h2>
<p>Donec imperdiet, lectus vestibulum sagittis tempus, turpis dolor euismod justo, vel tempus neque libero sit amet tortor. Nam cursus commodo tincidunt.</p>
<h3 id="subsection-1"><span class="header-section-number">4.2.1</span> Subsection 1</h3>
<p>This is the first part of the methodology. Duis tempor sapien sed tellus ultrices blandit. Sed porta mauris tortor, eu vulputate arcu dapibus ac. Curabitur sodales at felis efficitur sollicitudin. Quisque at neque sollicitudin, mollis arcu vitae, faucibus tellus.</p>
<h3 id="subsection-2"><span class="header-section-number">4.2.2</span> Subsection 2</h3>
<p>This is the second part of the methodology. Sed ut ipsum ultrices, interdum ipsum vel, lobortis diam. Curabitur sit amet massa quis tortor molestie dapibus a at libero. Mauris mollis magna quis ante vulputate consequat. Integer leo turpis, suscipit ac venenatis pellentesque, efficitur non sem. Pellentesque eget vulputate turpis. Etiam id nibh at elit fermentum interdum.</p>
<!-- 
Comments can be added like this.
-->
<h2 id="results-3"><span class="header-section-number">4.3</span> Results</h2>
<p>These are the results. In vitae odio at libero elementum fermentum vel iaculis enim. Nullam finibus sapien in congue condimentum. Curabitur et ligula et ipsum mollis fringilla.</p>
<h2 id="discussion-3"><span class="header-section-number">4.4</span> Discussion</h2>
<p>Figure  shows how to add a figure. Donec ut lacinia nibh. Nam tincidunt augue et tristique cursus. Vestibulum sagittis odio nisl, a malesuada turpis blandit quis. Cras ultrices metus tempor laoreet sodales. Nam molestie ipsum ac imperdiet laoreet. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
<!-- 
Figures can be added with the following syntax:
![my_caption \label{my_label}](source/figures/my_image.pdf){ width=50% }

For details on setting attributes like width and height, see:
http://pandoc.org/MANUAL.html#extension-link_attributes
-->
<figure>
<embed src="source/figures/example_figure.pdf" style="width:100.0%" /><figcaption>RV Calypso is a former British Royal Navy minesweeper converted into a research vessel for the oceanographic researcher Jacques-Yves Cousteau. It was equipped with a mobile laboratory for underwater field research. </figcaption>
</figure>
<h2 id="conclusion"><span class="header-section-number">4.5</span> Conclusion</h2>
<p>This is the conclusion to the chapter. Quisque nec purus a quam consectetur volutpat. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In lorem justo, convallis quis lacinia eget, laoreet eu metus. Fusce blandit tellus tellus. Curabitur nec cursus odio. Quisque tristique eros nulla, vitae finibus lorem aliquam quis. Interdum et malesuada fames ac ante ipsum primis in faucibus.</p>
<h1 id="research-containing-a-table"><span class="header-section-number">5</span> Research containing a table</h1>
<h2 id="introduction-4"><span class="header-section-number">5.1</span> Introduction</h2>
<p>This is the introduction. Phasellus non purus id mauris aliquam rutrum vitae quis tellus. Maecenas rhoncus ligula nulla, fringilla placerat mi consectetur eu. Aenean nec metus ac est ornare posuere. Nunc ipsum lacus, gravida commodo turpis quis, rutrum eleifend erat. Pellentesque id lorem eget ante porta tincidunt nec nec tellus.</p>
<h2 id="method-1"><span class="header-section-number">5.2</span> Method</h2>
<p>Vivamus consectetur, velit in congue lobortis, massa massa lacinia urna, sollicitudin semper ipsum augue quis tortor. Donec quis nisl at arcu volutpat ultrices. Maecenas ex nibh, consequat ac blandit sit amet, molestie in odio. Morbi finibus libero et nisl dignissim, at ultricies ligula pulvinar.</p>
<h3 id="subsection-1-1"><span class="header-section-number">5.2.1</span> Subsection 1</h3>
<p>This is the first part of the methodology. Integer leo erat, commodo in lacus vel, egestas varius elit. Nulla eget magna quam. Nullam sollicitudin dolor ut ipsum varius tincidunt. Duis dignissim massa in ipsum accumsan imperdiet. Maecenas suscipit sapien sed dui pharetra blandit. Morbi fermentum est vel quam pretium maximus.</p>
<h3 id="subsection-2-1"><span class="header-section-number">5.2.2</span> Subsection 2</h3>
<p>This is the second part of the methodology. Nullam accumsan condimentum eros eu volutpat. Maecenas quis ligula tempor, interdum ante sit amet, aliquet sem. Fusce tellus massa, blandit id tempus at, cursus in tortor. Nunc nec volutpat ante. Phasellus dignissim ut lectus quis porta. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
<!-- 
Comments can be added like this.
-->
<h2 id="results-4"><span class="header-section-number">5.3</span> Results</h2>
<p>Table  shows us how to add a table. Integer tincidunt sed nisl eget pellentesque. Mauris eleifend, nisl non lobortis fringilla, sapien eros aliquet orci, vitae pretium massa neque eu turpis. Pellentesque tincidunt aliquet volutpat. Ut ornare dui id ex sodales laoreet.</p>
<!-- Force the table onto a newpage -->

<table style="width:89%;">
<caption>This is the table caption. Suspendisse blandit dolor sed tellus venenatis, venenatis fringilla turpis pretium. </caption>
<colgroup>
<col style="width: 27%" />
<col style="width: 33%" />
<col style="width: 27%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Column 1</th>
<th style="text-align: left;">Column 2</th>
<th style="text-align: left;">Column 3</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Row 1</td>
<td style="text-align: left;">0.1</td>
<td style="text-align: left;">0.2</td>
</tr>
<tr class="even">
<td style="text-align: left;">Row 2</td>
<td style="text-align: left;">0.3</td>
<td style="text-align: left;">0.3</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Row 3</td>
<td style="text-align: left;">0.4</td>
<td style="text-align: left;">0.4</td>
</tr>
<tr class="even">
<td style="text-align: left;">Row 4</td>
<td style="text-align: left;">0.5</td>
<td style="text-align: left;">0.6</td>
</tr>
</tbody>
</table>
<h2 id="discussion-4"><span class="header-section-number">5.4</span> Discussion</h2>
<p>This is the discussion. Etiam sit amet mi eros. Donec vel nisi sed purus gravida fermentum at quis odio. Vestibulum quis nisl sit amet justo maximus molestie. Maecenas vitae arcu erat. Nulla facilisi. Nam pretium mauris eu enim porttitor, a mattis velit dictum. Nulla sit amet ligula non mauris volutpat fermentum quis vitae sapien.</p>
<h2 id="conclusion-1"><span class="header-section-number">5.5</span> Conclusion</h2>
<p>This is the conclusion to the chapter. Nullam porta tortor id vehicula interdum. Quisque pharetra, neque ut accumsan suscipit, orci orci commodo tortor, ac finibus est turpis eget justo. Cras sodales nibh nec mauris laoreet iaculis. Morbi volutpat orci felis, id condimentum nulla suscipit eu. Fusce in turpis quis ligula tempus scelerisque eget quis odio. Vestibulum et dolor id erat lobortis ullamcorper quis at sem.</p>
<h1 id="final-research-study"><span class="header-section-number">6</span> Final research study</h1>
<h2 id="introduction-5"><span class="header-section-number">6.1</span> Introduction</h2>
<p>This is the introduction. Nunc lorem odio, laoreet eu turpis at, condimentum sagittis diam. Phasellus metus ligula, auctor ac nunc vel, molestie mattis libero. Praesent id posuere ex, vel efficitur nibh. Quisque vestibulum accumsan lacus vitae mattis.</p>
<h2 id="method-2"><span class="header-section-number">6.2</span> Method</h2>
<p>In tincidunt viverra dolor, ac pharetra tellus faucibus eget. Pellentesque tempor a enim nec venenatis. Morbi blandit magna imperdiet posuere auctor. Maecenas in maximus est.</p>
<h3 id="subsection-1-2"><span class="header-section-number">6.2.1</span> Subsection 1</h3>
<p>This is the first part of the methodology. Praesent mollis sem diam, sit amet tristique lacus vulputate quis. Vivamus rhoncus est rhoncus tellus lacinia, a interdum sem egestas. Curabitur quis urna vel quam blandit semper vitae a leo. Nam vel lectus lectus.</p>
<h3 id="subsection-2-2"><span class="header-section-number">6.2.2</span> Subsection 2</h3>
<p>This is the second part of the methodology. Aenean vel pretium tortor. Aliquam erat volutpat. Quisque quis lobortis mi. Nulla turpis leo, ultrices nec nulla non, ullamcorper laoreet risus.</p>
<!-- 
Comments can be added like this.
-->
<h2 id="results-5"><span class="header-section-number">6.3</span> Results</h2>
<p>These are the results. Curabitur vulputate nisl non ante tincidunt tempor. Aenean porta nisi quam, sed ornare urna congue sed. Curabitur in sapien justo. Quisque pulvinar ullamcorper metus, eu varius mauris pellentesque et. In hac habitasse platea dictumst. Pellentesque nec porttitor libero. Duis et magna a massa lacinia cursus.</p>
<h2 id="discussion-5"><span class="header-section-number">6.4</span> Discussion</h2>
<p>This is the discussion. Curabitur gravida nisl id gravida congue. Duis est nisi, sagittis eget accumsan ullamcorper, semper quis turpis. Mauris ultricies diam metus, sollicitudin ultricies turpis lobortis vitae. Ut egestas vehicula enim, porta molestie neque consectetur placerat. Integer iaculis sapien dolor, non porta nibh condimentum ut.</p>
<h2 id="conclusion-2"><span class="header-section-number">6.5</span> Conclusion</h2>
<p>This is the conclusion to the chapter. Nulla sed condimentum lectus. Duis sed tempor erat, at cursus lacus. Nam vitae tempus arcu, id vestibulum sapien. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
<h1 id="conclusion-3"><span class="header-section-number">7</span> Conclusion</h1>
<!-- 
A chapter that concludes the thesis by summarising the learning points
and outlining future areas for research
-->
<h2 id="thesis-summary"><span class="header-section-number">7.1</span> Thesis summary</h2>
<p>In summary, pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nunc eleifend, ex a luctus porttitor, felis ex suscipit tellus, ut sollicitudin sapien purus in libero. Nulla blandit eget urna vel tempus. Praesent fringilla dui sapien, sit amet egestas leo sollicitudin at.</p>
<h2 id="future-work"><span class="header-section-number">7.2</span> Future work</h2>
<p>There are several potential directions for extending this thesis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam gravida ipsum at tempor tincidunt. Aliquam ligula nisl, blandit et dui eu, eleifend tempus nibh. Nullam eleifend sapien eget ante hendrerit commodo. Pellentesque pharetra erat sit amet dapibus scelerisque.</p>
<p>Vestibulum suscipit tellus risus, faucibus vulputate orci lobortis eget. Nunc varius sem nisi. Nunc tempor magna sapien, euismod blandit elit pharetra sed. In dapibus magna convallis lectus sodales, a consequat sem euismod. Curabitur in interdum purus. Integer ultrices laoreet aliquet. Nulla vel dapibus urna. Nunc efficitur erat ac nisi auctor sodales.</p>
<h1 id="appendix-1-some-extra-stuff" class="unnumbered">Appendix 1: Some extra stuff</h1>
<!-- 
This could be a list of papers by the author for example 
-->
<p>Add appendix 1 here. Vivamus hendrerit rhoncus interdum. Sed ullamcorper et augue at porta. Suspendisse facilisis imperdiet urna, eu pellentesque purus suscipit in. Integer dignissim mattis ex aliquam blandit. Curabitur lobortis quam varius turpis ultrices egestas.</p>
<h1 id="appendix-2-some-more-extra-stuff" class="unnumbered">Appendix 2: Some more extra stuff</h1>
<!-- 
This could include extra figures or raw data
-->
<p>Add appendix 2 here. Aliquam rhoncus mauris ac neque imperdiet, in mattis eros aliquam. Etiam sed massa et risus posuere rutrum vel et mauris. Integer id mauris sed arcu venenatis finibus. Etiam nec hendrerit purus, sed cursus nunc. Pellentesque ac luctus magna. Aenean non posuere enim, nec hendrerit lacus. Etiam lacinia facilisis tempor. Aenean dictum nunc id felis rhoncus aliquam.</p>

<!-- 
Do not edit this page.

References are automatically generated from the BibTex file (References.bib)

...which you should create using your reference manager.
-->
<h1 id="references-3" class="unnumbered">References</h1>
<div id="refs" class="references">
<div id="ref-Cousteau1963">
<p>Cousteau Jacques &amp; Dugan James, 1963. <em>The Living Sea: by Jacques-Yves Cousteau</em>, London: Hamish Hamilton.</p>
</div>
</div>
            </body>
</html>

