# List of figures {.unnumbered}

\setlength{\parskip}{0pt}

\setcounter{page}{7}
\renewcommand{\thepage}{\arabic{page}}

**Figure 2.1** \Dotfill pp

> Synchronizing Dialdehyde-Tissue-Gel Formation Enables Scalable Tissue Preservation

**Figure 2.2** \Dotfill pp

> SWITCH and Co-registration Algorithms Enable Highly Multiplexed Imaging at Single-Cell Resolution

**Figure 2.3** \Dotfill pp

> SWITCH Enables Proteomic Imaging and High-Dimensional Quantitative Phenotyping of Human Clinical Samples

**Figure 2.4** \Dotfill pp

> SWITCH Enables Simple, Rapid, and Scalable Tissue Clearing

**Figure 2.5** \Dotfill pp

> SWITCH Enables Visualization and Quantitative Analysis of Entire Myelinated Fiber Tracts

**Figure 2.6** \Dotfill pp

> SWITCH Increases Uniformity of Antibody Labeling in Thick Tissues

**Figure 3.1** \Dotfill pp

> Magnified and accessible 3D proteome library of whole intact organs

**Figure 3.2** \Dotfill pp

> Comparison of multiscale architectures before and after MAP processing

**Figure 3.3** \Dotfill pp

> Super-resolution imaging showing the 3D proteome library and subcellular details of MAP-processed intact tissue

**Figure 3.4** \Dotfill pp

> Multiplexed staining of MAP-processed tissue

**Figure 3.5** \Dotfill pp

> Intercellular connectivity and its reconstruction at single-fiber resolution in MAP-processed tissue

**Figure 3.6** \Dotfill pp

> Immunolabeling and imaging of thick MAP-processed tissues

**Figure 4.1** \Dotfill pp

> eFLASH enables rapid, uniform, and cost-efficient labeling of organ-scale tissues

**Figure 4.2** \Dotfill pp

> Single eFLASH protocol enables complete and uniform staining of various tissue types with a wide range of molecular probes

**Figure 4.3** \Dotfill pp

> Quantitative brain-wide cell type mapping

**Figure 4.4** \Dotfill pp

> Brain-wide comparison of genetic cell-type labeling and eFLASH-driven protein-based cell type labeling

**Figure 4.5** \Dotfill pp

> Multidimensional analysis of an eFLASH-stained marmoset brain block at single cell resolution

**Figure 5.1** \Dotfill pp

> Overall image processing pipeline for whole brain analysis

**Figure 5.2** \Dotfill pp

> Destriping of light-sheet microscopy images using pystripe

**Figure 5.3** \Dotfill pp

> Stitching and illumination correction of light-sheet microscopy images using TSV

**Figure 5.4** \Dotfill pp

> Atlas alignment and region-based fluorescence quantification using nuggt

**Figure 9.1** \Dotfill pp

> SWITCH Antibody Compatibility

**Figure 9.2** \Dotfill pp

> Multi-color Imaging of SWITCH-Processed Samples

**Figure 9.3** \Dotfill pp

> Autocorrelation Analysis of Myelinated Fibers

**Figure 9.4** \Dotfill pp

> Autocorrelation Analysis of Bulk Myelinated Fibers

**Figure 9.5** \Dotfill pp

> GA-Induced Fluorescence

**Figure 9.6** \Dotfill pp

> mRNA FISH on Thermally Cleared Samples

**Figure 9.7** \Dotfill pp

> Expansion of various organs with MAP

**Figure 9.8** \Dotfill pp

> Validation of commercial antibodies targeting cell-type markers in MAP-processed tissues

**Figure 9.9** \Dotfill pp

> Validation of commercial antibodies targeting neurofilament markers in MAP-processed tissues

**Figure 9.10** \Dotfill pp

> Antibody signals in the presence of sodium deoxycholate (NaDC)

**Figure 9.11** \Dotfill pp

> Measurement of NaDC concentration and pH of eFLASH buffers

**Figure 9.12** \Dotfill pp

> Cell detection algorithm

**Figure 9.13** \Dotfill pp

> Representative NPY+ neurons with different dendrite polarities

\setlength{\parskip}{9pt}
\newpage
