# List of figures {.unnumbered}

\setlength{\parskip}{0pt}

\setcounter{page}{7}
\renewcommand{\thepage}{\arabic{page}}

**Figure 2.1** \Dotfill 27

> Synchronizing Dialdehyde-Tissue-Gel Formation Enables Scalable Tissue Preservation

**Figure 2.2** \Dotfill 31

> SWITCH and Co-registration Algorithms Enable Highly Multiplexed Imaging at Single-Cell Resolution

**Figure 2.3** \Dotfill 33

> SWITCH Enables Proteomic Imaging and High-Dimensional Quantitative Phenotyping of Human Clinical Samples

**Figure 2.4** \Dotfill 36

> SWITCH Enables Simple, Rapid, and Scalable Tissue Clearing

**Figure 2.5** \Dotfill 38

> SWITCH Enables Visualization and Quantitative Analysis of Entire Myelinated Fiber Tracts

**Figure 2.6** \Dotfill 41

> SWITCH Increases Uniformity of Antibody Labeling in Thick Tissues

**Figure 3.1** \Dotfill 55

> Magnified and accessible 3D proteome library of whole intact organs

**Figure 3.2** \Dotfill 57

> Comparison of multiscale architectures before and after MAP processing

**Figure 3.3** \Dotfill 59

> Super-resolution imaging showing the 3D proteome library and subcellular details of MAP-processed intact tissue

**Figure 3.4** \Dotfill 61

> Multiplexed staining of MAP-processed tissue

**Figure 3.5** \Dotfill 63

> Intercellular connectivity and its reconstruction at single-fiber resolution in MAP-processed tissue

**Figure 3.6** \Dotfill 65

> Immunolabeling and imaging of thick MAP-processed tissues

**Figure 4.1** \Dotfill 82

> eFLASH enables rapid, uniform, and cost-efficient labeling of organ-scale tissues

**Figure 4.2** \Dotfill 85

> Single eFLASH protocol enables complete and uniform staining of various tissue types with a wide range of molecular probes

**Figure 4.3** \Dotfill 87

> Quantitative brain-wide cell type mapping

**Figure 4.4** \Dotfill 89

> Brain-wide comparison of genetic cell-type labeling and eFLASH-driven protein-based cell type labeling

**Figure 4.5** \Dotfill 91

> Multidimensional analysis of an eFLASH-stained marmoset brain block at single cell resolution

**Figure 5.1** \Dotfill 112

> Overall image processing pipeline for whole brain analysis

**Figure 5.2** \Dotfill 116

> Destriping of light-sheet microscopy images using pystripe

**Figure 5.3** \Dotfill 118

> Stitching and illumination correction of light-sheet microscopy images using TSV

**Figure 5.4** \Dotfill 119

> Atlas alignment and region-based fluorescence quantification using nuggt

**Figure 6.1** \Dotfill 132

> Volumetric image preprocessing and compression

**Figure 6.2** \Dotfill 133

> Coarse registration based on morphological alignment

**Figure 6.3** \Dotfill 135

> Nuclei matching and thin-plate spline warping

**Figure 6.4** \Dotfill 136

> Demonstration of 3D coregistration of a whole mouse hemisphere at single-cell resolution

**Figure 6.5** \Dotfill 138

> Demonstration of 3D coregistration of a large block of marmoset visual cortex at single-cell resolution

**Figure 7.1** \Dotfill 148

> Pipeline for intact organoid analysis

**Figure 7.2** \Dotfill 150

> Single cell detection and analysis

**Figure 7.3** \Dotfill 152

> Ventricle segmentation and cytoarchitecture phenotyping

**Figure 7.4** \Dotfill 154

> Whole organoid analysis for unbiased quantitative studies

**Figure 7.5** \Dotfill 156

> Hyper-dimensional analysis of multiscale changes during cerebral organoid maturation

**Figure 7.6** \Dotfill 158

> Hyper-dimensional analysis of multiscale changes caused by Zika virus infection

**Figure 7.7** \Dotfill 160

> Hyper-dimensional analysis of multiscale changes in different culture protocols

**Figure 8.1** \Dotfill 176

> Construction of a high-resolution two-photon SLA printer

**Figure 8.2** \Dotfill 177

> Two-photon SLA of micro-vessels from biocompatible materials

**Figure 8.3** \Dotfill 179

> Tiling strategy for printing large-scale vascular systems

**Figure 8.4** \Dotfill 180

> Direct printing of microporous structures improves mass transport from printed reservoirs

**Figure 8.5** \Dotfill 181

> Modulation of microfluidic transport properties by direct fabrication of engineered pore structures

**Figure 8.6** \Dotfill 182

> Integration of vasculature into cerebral organoids shows mature neurons micro-vessel integration

**Figure 9.1** \Dotfill 217

> SWITCH Antibody Compatibility

**Figure 9.2** \Dotfill 218

> Multi-color Imaging of SWITCH-Processed Samples

**Figure 9.3** \Dotfill 219

> Autocorrelation Analysis of Myelinated Fibers

**Figure 9.4** \Dotfill 220

> Autocorrelation Analysis of Bulk Myelinated Fibers

**Figure 9.5** \Dotfill 221

> GA-Induced Fluorescence

**Figure 9.6** \Dotfill 222

> mRNA FISH on Thermally Cleared Samples

**Figure 9.7** \Dotfill 236

> Expansion of various organs with MAP

**Figure 9.8** \Dotfill 237

> Validation of commercial antibodies targeting cell-type markers in MAP-processed tissues

**Figure 9.9** \Dotfill 238

> Validation of commercial antibodies targeting neurofilament markers in MAP-processed tissues

**Figure 9.10** \Dotfill 240

> Antibody signals in the presence of sodium deoxycholate (NaDC)

**Figure 9.11** \Dotfill 242

> Measurement of NaDC concentration and pH of eFLASH buffers

**Figure 9.12** \Dotfill 243

> Cell detection algorithm

**Figure 9.13** \Dotfill 244

> Representative NPY+ neurons with different dendrite polarities

**Figure 9.14** \Dotfill 258

> Optimization of aldehyde fixation for organoids

**Figure 9.15** \Dotfill 259

> Optimization of SHIELD protocol for organoids

**Figure 9.16** \Dotfill 260

> Nuclei detection and segmentation accuracy

**Figure 9.17** \Dotfill 261

> Ventricle segmentation and U-Net model validation

\setlength{\parskip}{9pt}
\newpage
